% Chapter 01
\chapter{Introduction}\label{ch:intro}
As the most abundant protein in most eukaryotic cells, actin participates in more protein-protein interactions than any known protein. Additionally actin is highly conserved among organisms and even among the three isoforms in vertebrates, only varies by a few amino acids near the N-terminus \cite{dominguez_actin_2011}. 
The actin cytoskeleton is made up of various dynamic networks made up of filamentous actin (F-actin) and other actin binding proteins (ABPs). These networks are formed and degraded at certain times and locations within cells and are important for motility, cytokinesis, adherence, and %blank. 

\section{Actin}\label{actin-intro}

Actin is a 46 kDa globular protein and contains four subdomains. Subdomains 1 and 3 are structurally similar and subdomains 1 and 2 form the outer domain while 3 and 4 form the inner, larger domain. Between these domains is a hinge region and two large clefts, a nucleotide binding cleft and a target binding cleft. The nucleotide binding cleft binds ATP and an associated cation (Mg\textsubscript{2+} in cells). Many proteins bind actin in the target-binding cleft \citep{dominguez_actin_2009}.

Actin assembles into polar filaments with new subunits being primarily added onto the barbed end (subdomains 1 and 3) versus the pointed end (subdomains 2 and 4). An actin filament contains two protofilaments that wrap around each other with a right-handed twist along the long-axis (hanson J the strucutre of F-actin 1963). However, the repeated unit of symmetry is a left-handed helix with \mytilde13 actin subunits repeating every 35.9nm. The G-actin to F-actin transition consists of a 12-13$^\circ$ rotation of the outer domain (subunits 1 and 2) with respect to the inner domain (subunits 3 and 4) with some additional bending movements of subdomains 2 and 4. Therefore, F-actin is flatter than F-actin and the D-loop in subdomain 2 inserts into the target-binding cleft of the subunit above \citep{dominguez_actin_2011}.

The formation of an actin filament contains two main steps, nucleation and elongation. 
Nucleation is slow, forming a seed
Elongation is dependent on concentration
In vitro, ~10 sub/s/uM
Contain two protofilaments 
Main binding cleft
Many proteins bind the same cleft on actin subunits

\section{Actin networks}\label{network-intro}

Cell division, ect from introduction that is not main focus (or at end)
Lamellipodia
Edge of motile cells, dense branched network nucleated by Arp2/3 complex
Capped by capping protein
Filopodia
Fingerlike projections that emerge from the lamellipodium. Important for exploration of extracellular space in motile cells. 
Convergent Elongation
Formin and Ena/VASP: different dynamics and lengths
Stress Fibers/Focal adhesions
Different types of stress fibers
Why?
Focal adhesions form and then release from forward to backward in motile cells

\section{Actin binding proteins}\label{abps} 
% sub sub sections within here?
Profilin is a small actin binding protein. Slows down nucleation and elongation?
Amount of actin in cells that is bound to profilin?
Tropomyosin here?
\subsection{Nucleation factors}\label{nucleators}
different classes
Arp2/3 Complex
Complex made of seven subunits, with Arp2 and Arp3 subunits similar to actin
Binds to the sides of exisiting actin filaments (mother filaments) and nucleates a new daughter branch. The branch forms at a consistent 70 degree angle.
Found in lamellipodium, actin patches?
Formin, uses profilin-actin to nucleate, more later
   mechanism here of how formin nucleates
   unless in review
VopL/F and WH2
Newer class of nucleators
Rely on multiple WH2 domains that can bind to G-actin monomers. 
Vops from cholera, mechanism was debated between barbed or pointed end binding
\subsection{Elongation factors}\label{elongators}
Formin also processively elongates actin
Ena/VASP, tetrameric actin elongation factor that processively associates with barbed ends of actin filaments more later 
\subsection{Bundling proteins}\label{bundlers}
(figure here)
Different families
CH domain
Alpha actinin, fimbrin/plastin, filamin?
Fascin
Different architectures
Packing
Transverse repeat
Spacing
Polarity
Different localization in cells (networks next or first?)
Do this one protein at a time?

\section{Ena/VASP and Formins}\label{ena-formin-review}

\subsection{Comparison of processive actin elongation factors}

Why are we interested in learning the mechanism of abps?
Many important cellular functions depend on the architecture of a dynamic actin cytoskeleton forming at the correct location and time during the cell cycle. Cellular division, motility, and endocytosis are a few examples of processes in which actin filaments must be nucleated, polymerized, severed, and depolymerized at the precise time and location in the cell. Here I will focus on the mechanism of two processive protein families, formins and Ena/VASP, which bind to the barbed end of actin filaments and affect actin filament polymerization. Interestingly, though these two families have different structures and use distinct mechanisms, they share similarities in how they increase actin polymerization as well as that they are both localized to the leading edge and involved in filopodia formation. However, their mechanistic differences in terms of processive run lengths, diverse protein binding partners, and formin's reliance on profilin actin bring up many interesting questions concerning how these proteins are utilized by a cell within different and even the same processes. The knowledge of formin's mechanism is a step ahead of Ena/VASP's, exploring the role of force, rotation, and diverse actin regulatory domains on its actin assembly properties. However, this allows insight to compare and contrast the two processive machines and understand how they both individually and concurrently help to assemble the necessary actin networks for proper cellular function. 

\subsection{Cellular processes}\label{ena-formin-cellular-processes}

\subsubsection{Ena/VASP}

Ena/VASP processes: actin polymerization
Bundling and nucleation in vitro, not sure in vivo implications, cell migration?
Vasodilator-stimulated phosphoprotein (VASP) was first discovered in human platelets as a substrate of both the cAMP- and cGMP-dependent protein kinases \citep{halbrugge_analysis_1990}. Simultaneously, enabled (ena) was found in Drosophila during a genetic screen for surpressors of abelson protein tyrosine kinase mutant phenotypes \citep{gertler_genetic_1990}. Ena and VASP were found to be homologous \citep{ahern-djamali_identification_1999} and homologs have been found in all multicellular metazoan cells and Dictyostelium \citep{sebe-pedros_insights_2013}.
Invertebrates (Drosophila, C.elegans, Dictyostelium ect.) only have one isoform of Ena/VASP while mammalians have three: Mena (mammalian Enabled), VASP, and EVL (Ena-Vasp-like) \cite{gertler_mena_1996}.
Ena/VASP localization: lamellipodia, filopodia, stress fibers, and focal adhesions adherence junctions

\subsubsection{Formin}

Formin homology proteins (formins) are highly conserved actin binding proteins that function in multiple cellular processes such as cytokinesis, oogenesis, and stress fiber and filopodia formation. Furthermore, formins can interact with both actin and microtubules [cite], allowing for communication between these two cytoskeleton systems. Formins were first discovered as mutations at the mouse limb deformity (ld) locus resulting in development defects \citep{woychik_formins:_1990}. Since their discovery the formin family has grown, now counting 15 different formins in mammalian cells. The formins are localized to different areas of the cell including the leading edge, tips of filopodia, cell-cell junctions, and cytokinetic rings. As a class, formins are implicated in nucleation of nascent actin filaments, processive elongation, and competition with capping protein as well as other F-actin barbed end binding proteins. However, there are some proteins within the formin family (i.e. INF2) that have been shown to carry out additional behaviors, such as depolymerization, bundling of filaments, severing, or actin monomer binding \citep{gurel_assembly_2015}. Though the mechanism of formin-mediated processive filament assembly has been well-studied, the individual characteristics of different formins and how those characteristics influence their distinct roles within the same cell  continue to be open questions. 

\subsection{Domain organization}\label{ena-formin-domains}

\subsubsection{Ena/VASP}
N-terminal Ena-VASP homology 1 (EVH1) domain: localization, FP4 binding (FPPPP) such as formin, zyxin, lamellipodin, vinculin [ball dual 2000, niebuhr a novel 1997 klostermann 2000].
EVH1 present in all Ena/VASP family members as well as distantly related Wiskott-Aldrich syndrome proteins, WASP and N-WASP, and Homer/Vesl family, though proteins from outside the Ena/VASP family recognize different domains. Structurally the EVH1 domain is similar to pleckstrin homology (PH) and phosphotyrosine-binding (PTB) domains though there is little sequence similarity to these domains \citep{prehoda_structure_1999,reinhard_actin-based_2001} [fedorov structure, ball dual 2000].
The C-termianl Ena/VASP Homology 2 (EVH2) domain contains the main actin binding domains. The G-actin binding domain (GAB) is related to a WH2 domain which also binds actin monomers. The F-actin binding domain (FAB) and the C-terminal coiled-coil facilitates tetramerization. Interestingly, it was found that mammalian proteins can form hetero-tetramers in vitro [gertler]. 
Between the EVH1 and EVH2 domain is a poly-proline region. This region can bind to profilin and other SH3 containing molecules such as Abl, Src [lanier from 2000] \citep{gertler_mena_1996}.

\subsubsection{Formin}
The classical formin homology 1 (FH1) and formin homology 2 (FH2) domains are the characteristic  domains of the formin family. The N-terminus of formin contains regulatory and localization domains that vary depending on the subfamily of formins. The largest subfamily, Diaphanous-related formins (DRF), is autoregulated by the N-terminal Diaphanous inhibitory domain (DID) binding to a C-terminal Diaphanous autoregulatory domain (DAD).  Rho-GTPase binding to the N-terminal GTPase binding domain (GBD) can relieve this autoregulation. Subsets outside of DRFs have C-terminal DAD or Wiskott-Aldrich syndrome homology region 2 (WH2) domains. Other domains are found across the diverse formin family, including those that function in auto-regulation, inhibition, localization, depolymerization, and filament actin (F-actin) binding.

The FH1 and FH2 domains have canonically been associated with the formins' actin assembly properties and dimerization. The FH1 domain contains between one to 15 polyproline repeats that are able to bind to profilin, but with varying affinity. This domain is flexible and allows for increased actin filament elongation in the presence of profilin-actin likely by increasing the local concentration of actin monomers near the polymerizing barbed end (ref). The FH2 domain forms a head-to-tail dimer that is thought to stabilize an actin dimer, promoting nucleation of a nascent actin filament. The dimerized FH2 domains can additionally encircle and remain associated with the barbed end of an actin filament, facilitating the addition of profilin-actin to the barbed end during elongation. For a detailed review of the FH1 and FH2 domains, see \citep{paul_review_2009}. 

\subsection{Mechanism of processive F-actin elongation}\label{ena-formin mechanism}

\subsubsection{Ena/VASP}
G-actin or profilin-actin transfered to barbed end. Change in conformation of G-actin once it adds to the filament allows for release of newly added monomer. 
Doesn't fall off like formins with a certain number of steps. Actin concentration independent \citep{hansen_vasp_2010}.
Rotation not known
All arms equal?
Clustering in vivo?

\subsubsection{Formin}

There are two main models for formin processive elongation, stair stepping and stepping second. Both models incorporate an over-rotation of the short-pitch helical twist at the barbed end from native F-actin state of $167^{\circ}$ to $180^{\circ}$ when FH2 is tightly bound. FH2 exists in a dynamic equilibrium between the tightly bound “closed” state ($180^{\circ}$) and loosely bound “open” state ($167^{\circ}$). The closed state does not allow addition of new monomer because the filament is over rotated and does not present favorable contacts for an incoming subunit \citep{otomo_structural_2005}. The initial model, stair stepping, describes the FH2 bound tightly to three actin subunits in the closed state. The FH2 domain can then transition to an open state by releasing one actin binding contact. This allows a monomer to bind to the barbed end and FH2. With the addition of a new monomer, the FH2 can return back to the closed state. Therefore, the FH2 domain translocates first, then facilitates monomer addition \citep{otomo_structural_2005}. 

The stepping second model reverses these steps and proposes that monomer addition promotes a translocation. Importantly, the translocation state is the most loosely bound state where formin would dissociate. In this model, the FH2 domain again begins in the closed state with four actin-binding contacts. The FH2 domain transitions to an open state that is permissive to monomer addition. After monomer addition, the formin is transiently bound to two interior subunits. The FH2 can then move onto the new barbed end by passing through an unstable state (dissociative) state to again reach its closed low energy state \citep{paul_role_2008}.

The stair stepping model postulates that formin enters its most unstably bound (dissociative) state independent of monomer addition. The second stepping model predicts that formin only enters the dissociative state if a monomer is added. Therefore the second stepping model predicts that the dissociation rate should be proportional to the number of subunits added whereas the stair stepping model predicts that dissociation should be a function of time \citep{paul_review_2009}. In support of the second stepping model, formin dissociation rate appears to be a function of the number of subunits added \citep{paul_role_2008}. 

Whether the FH2 domain is in a closed or open state can dictate whether additional monomers can be added to the barbed end of a filament. Modulating the open-closed equilibrium state would allow fine-tuning of formin-mediated elongation rates. Using microfluidics, recent studies have shown that formin-mediated elongation rate is sensitive to tension on the formin and filament \citep{courtemanche_tension_2013,jegou_formin_2013}. Jégou et al. showed that the formin mDia1's elongation rate is increased up to two-fold when under tension. Similarly, Courtemanche et al. have shown that the elongation rate of the formin Bni1 also increases when a pulling force is applied, though only in the presence of profilin. Both groups suggest that a pulling force shifts the gating equilibrium of the FH2 domain towards the open state, which then causes an increase in elongation. Interestingly, when the FH1 domains are also pulled straight the elongation rate still increases. Therefore, the space that the FH1 can explore to find monomers is not a major factor because when the FH1 have reduced mobility you do not see reduced elongation rates.  The sensitivity to tension opens up the possibility that within cells formins can sense mechanotransduction of forces, which could tune formin elongation rates.

These models also suggest that formins rotate to track the helical twist of the actin filament as it processively elongates filaments. New tools have been used to gain a clearer understanding of formin rotation during elongation. Previously, it had been suggested that formins do not rotate while processively binding to the barbed end of filaments because when formins are adhered to the surface, the filaments buckled and did not supercoil \citep{kovar_insertional_2004}. Recently Mizuno et al. used fluorescence polarization of a labeled filament with a formin (mDia1) biotinylated to the surface and were able to directly observe this rotation. The tetramethylrhodamine (TMR) label on the filament had periodic movement that corresponded to half of the long-pitch helical length of F-actin. Since the formin was linked to the surface, the filament was forced to rotate rather than the formin. In vivo it is likely that the formin rotates during filament elongation, especially in crosslinked F-actin networks \citep{mizuno_rotational_2011}. Rotation may be able to apply torque to actin filaments resulting in a change in twist, something actin-binding proteins may detect. Formins may be able to relieve this torque in various ways, either by rotating or by 'slipping' on the end of the filament. Alternatively, in some situations the filaments themselves may be able to rotate. However, filaments usually exist as part of highly connected F-actin networks, which would seem to inhibit filament rotation. This work raises many new questions including whether formin binding partners would rotate with the formins and if this would affect their actin assembly properties. It also opens new possibilities for mechanisms of regulation involving formin rotation. 

\subsection{Behaviors outside of polymerization}\label{ena-formin-behaviors}

\subsubsection{Ena/VASP}
Different homologs have stronger/weaker g-actin binding. Lead to differences in actin elongation
Profilin here
Capping protein competition
bundling
nucleation

\subsubsection{Formin}

The main behaviors of formins include F-actin nucleation, processive elongation, and actin filament crosslinking. There are a variety of ways that formins balance these behaviors to fulfill unique functions. For example, between the three different formin isoforms in fission yeast (Cdc12, Fus1, and For3) the nucleation rates, processivity, elongation and ability to bundle vary significantly. Cdc12 and Fus1 can efficiently nucleate filaments and are 70-fold better nucleators than For3 \citep{scott_functionally_2011}. Cdc12 and For3 are both highly processive and elongate filaments at a moderate rate, while Fus1 is only able to elongate filaments at half of this rate in the absence of profilin. Where Cdc12 adds 27 times more monomers than Fus1 on an average processive run, Fus1 is the only fission yeast formin that can bundle filaments. These varying behaviors of the different formin isoforms found within the same organism raises the question of how these formins are tuned to their unique role within the cell. Furthermore, how do proteins differentially interact with these formins, and how can that contribute to or change these properties?

The actin assembly activity of formins can also be tuned to the environment in the cell. Higashida et al. have shown that mDia1 rapidly increases processive F-actin assembly with the release of cell tension using sequential fluorescence decay after photoactivation (s-FDAPplus) to visualize the nucleation and elongation activity of mDia1 \citep{higashida_f-_2013}. How actin and binding proteins are able to sense cell stress and forces is not very well understood. It will be important to look into what protein(s) are initially sensing the forces on the cell and how this is translated to modifying actin networks. 
% include Dennis' research in this section

Previous formin research has often focused on the two characteristic FH1 and FH2 domains to understand their actin assembly properties. However, formins contain other domains that vary across the formin family, including those that function in auto-regulation, inhibition, depolymerization, and filament actin (F-actin) binding. New studies have shown that the C-terminal tails of different formins play an important role in its actin assembly and nucleation properties. Cappuccino (Capu) \citep{vizcarra_role_2014}, FMNL3 \citep{heimsath_c_2012}, and mDia1 \citep{gould_formin_2011} can bind G-actin monomers with their C-terminal tails to help the FH2 domains nucleate F-actin filaments. Furthermore, Vizcarra et al. suggest that Capu's C-terminal tail also enhances elongation by forming nonspecific electrostatic interactions with F-actin that can assist in stabilizing the open-state FH2 dimer binding to the actin filament. INF2 has been shown to have important domains present in the C-terminal tail that can bind and sequester actin monomers \citep{chhabra_inf2_2006}. However, even though many formins depend on their C-terminal tail for their specific function, they do not all function equally. Replacing the C-terminal tail of Capu with the C-terminal tail of DRF formins changed its nucleation and processivity properties \citep{vizcarra_role_2014}. The growing evidences of the role of the C-terminal tail in tuning of nucleation and processivity opens up the possibility of differential regulation and function of formin isoforms present in the same cell. Whether these C-terminal interactions have important contributions to formin function and regulation overriding that of the FH1FH2 is unclear. 

Beyond binding to actin, Capu's C-terminal tail along with specific residues of the FH2 have been shown to promote high affinity microtubule binding, though this occurs separately from Capu's actin assembly properties \citep{roth-johnson_interaction_2014}. Actin and microtubule dynamics are linked within the cell and the details of this relationship remain a current question in the field. Recently it has been shown that formins mDia1, mDia2, and mDia3 are involved in ErbB2-dependent microtubule capture through their FH2 domains \citep{daou_essential_2014}. Additionally, TIRFM was used to visualize the interaction between microtubule plus end localizing CLIP-170 and formin mDia1, which stimulates actin growth from microtubule plus ends \citep{henty-ridilla_accelerated_2016}. This opens up many new questions about how various formins could be regulating actin-microtubule crosstalk. 


\subsection{Regulation}\label{ena-formin-regulation}
phosphorylation
localization


\subsubsection{Formin}

It is well established that DRF formins are autoregulated by interaction of the DID-DAD domains, which can be relieved by Rho-GTPase binding. Recently, a structure has been solved of full-length mDia1 that confirms that the DID-CC domain sterically occludes where actin binds on FH2 during nucleation \citep{maiti_structure_2012}. Furthermore, it was shown that the formin does not quickly return to its autoinhibited state during processive elongation, which opens further questions about how, or if, formins are stopped once a processive run has been initiated. Other insights into autoregulation have been highlighted by the structure of Rho-GTPase Cdc42 with formins FMNL1 and FMNL2. The structure along with biochemical data suggests that specific interactions between formin FMNL2 and the Rho-GTPase insert helix are at play when determining specificity of Rho-GTPases to release formin autoinhibition \citep{kuhn_structure_2015}. Beyond autoregulation, it has been shown that PIP2 can inhibit mDia1 by binding to its C-terminal tail. Phospholipids can recruit mDia1 to the membrane \citep{van_gisbergen_class_2012}, which then causes an accumulation of PIP2 so phospholipids are able to regulate formin localization to the membrane and inactivation \citep{ramalingam_phospholipids_2010}. 

Recent studies focusing on formin binding partners has opened up even further opportunities for regulation. The formin mDia1 has been shown to interact with adenomatous polyposis coli (APC) through its tail, forming a complex that acts as a potent nucleator \citep{breitsprecher_rocket_2012,okada_adenomatous_2010}. Likewise, nucleation-promoting factor Bud6 is known to enhance the nucleation of formin Bni1 \citep{moseley_differential_2005}. Capu and closely related FMN2 have been shown to interact with the nucleating protein Spire with their C-terminal domains \citep{montaville_role_2016,montaville_spire_2014,pechlivanis_identification_2009,vizcarra_structure_2011}. Recent studies have visualized a 'decision complex' of capping protein and formins mDia1 and FMNL2. Previous genetic and biochemical work suggested that formins and capping protein were entirely antagonistic (Kovar, 2006)**, yet single-molecule TIRF microscopy has shown that both of these proteins can bind simultaneously to a barbed end for a set amount of time before one gains sole control \citep{bombardier_single-molecule_2015,shekhar_formin_2015}. Further questions remain about how different formins are regulated to act at the precise time and location within a cell. Exploring the interaction between a formin and different protein binding partners and macromolecules can open up new possibilities for regulation.

\subsection{Ena/VASP and formin interaction}\label{ena-formin-interaction}

Although their biochemical mechanisms and rate constants are quite different, Ena/VASP and formins both stimulate the assembly of long, straight filaments within cells in two ways: both families 1) protect actin filaments from capping protein by processive association with the barbed end and 2) increase elongation rate of filaments during this processive association. In line with these activities, both protein families induce the assembly of filopodia \citep{bilancia_enabled_2014,homem_exploring_2009} and localize to zones of actin assembly at distal tips of these structures. However, Ena/VASPs and formins appear to drive distinct types of filopodia \citep{barzik_ena/vasp_2014,bilancia_enabled_2014,nowotarski_actin_2014,homem_exploring_2009}. In general, formin-induced filopodia are much longer, and the actin is not highly connected with other actin networks. Conversely, Ena/VASP-induced filopodia are shorter and deeply rooted in lamellar networks. During dorsal closure in fly embryos, motile epithelial cells displayed filopodia more characteristic of Ena, while underlying non-motile amnioserosal cells displayed filopodia more characteristic of formin Dia \citep{nowotarski_actin_2014}. However, both proteins played roles in both tissue types. It is possible that in each tissue type, activities of each elongation factor are modulated accordingly to drive different functions. For example, it has been suggested that Ena/VASP may work primarily in reorganizing preexisting networks in the leading edge of motile cells through a convergent elongation-type mechanism \citep{svitkina_mechanism_2003}. Formin, which can also nucleate actin filaments, may not require a preexisting lamellipodial network and assemble its own filaments for filopodia de novo (Faix et al., 2009), and therefore may function more in non-motile cells. 

However, Ena/VASPs and formins directly bind to each other and colocalize at the distal tips of some filopodia. These observations led different groups to question why these proteins with generally similar assembly properties would colocalize in filopodia and how their direct binding might regulate each other's activity. Ena/VASP's EVH1 domain mediates this interaction either through formin's FH1 \citep{bilancia_enabled_2014} and/or FH2 domain (Barzik et al., 2014a; Schirenbeck et al., 2006). Ena's EVH1 domain binds to the fly formin Diaphanous with a Kd of 13 $\mu$M, similar to the low affinity EVH1-FPPP4 interaction that the EVH1 uses to bind a large number of other proteins \citep{prehoda_structure_1999}. This interaction may negatively regulate formin by interfering with the core assembly machinery. 

Recent data shows that in addition to Ena/VASP and formin driving distinct filopodial morphology and dynamics, Ena/VASP is necessary for proper function in some cells. Bilancia et al., found that Enabled could negatively regulate the formin Diaphanous both in vivo and in vitro. Work from the Gertler lab extended these ideas by showing the physiological consequences of driving filopodia formation with either formin or Ena/VASP \citep{barzik_ena/vasp_2005}. This study found that although formins can generate filopodia, Ena/VASP is necessary for initiating focal contacts and integrin-dependent signaling. It is not currently clear if these dynamics play out in all cell types, and will be an important area of future research. In addition, the effect of formin on Ena/VASP activity, if any, is also an area of future research.

\section{Single-molecule total internal reflection microscopy (TIRFM)}\label{tirfm}
The main experimental set-up that we have used to study actin binding proteins and their activity and dynamics is total internal reflection microscopy (TIRFM). This microscopy uses a special objective that allows for a low angle of incidence of the incoming laser. The laser internally reflects between the glass slide and coverslip. This internal reflection causes an evanescent wave that can illuminate fluorophores within 150 $\mu$m from the coverslip. By only illuminating this small area, we get much better signal to noise by reduction of background fluorescence beyond the 150 $\mu$. TIRFM is especially beneficial for observing single molecules due to the reduced background. Single-molecule TIRFM allows us to measure single events at low concentrations of our protein of interest. Specifically for studying the actin cytoskeleton, we are able to watch filaments assemble in real time and monitor binding dynamics and how actin binding proteins interact with F-actin. Compared to bulk actin assays such as fluorescent pyrene assays we are able to distinguish polymerization, nucleation, and measure actual elongation rates of the filaments. We are also able to localize where proteins are interacting with the actin filament as well as other actin binding proteins. One of the biggest advantages of visualizing actin binding proteins with TIRFM is the ability to measure dynamics of both binding and dissociation of the F-actin. This kinetic information can inform the molecular mechanisms of actin binding proteins. 


Lead-in to rest of thesis?
What is the molecular mechanism of Ena/VASP on single and bundled filaments?
How do bundling proteins effect other actin binding proteins?
What are the effect of WH2 containing molecules?
