% Chapter 02

\chapter{Ena/VASP processive elongation is modulated by avidity on actin filaments bundled by the filopodia crosslinker fascin}\label{ch:ena}

\section[Abstract]{Abstract\footnotemark}
Ena/VASP tetramers are processive actin elongation factors that localize to diverse F-actin networks composed of filaments bundled by different crosslinking proteins, such as filopodia (fascin), lamellipodia (fimbrin), and stress fibers ($\alpha$-actinin). Previously, we found that \textit{Drosophila} Ena takes \mytilde3-fold longer processive runs on trailing barbed ends of fascin-bundled actin filaments. Here, we used single-molecule TIRFM and developed a kinetic model to further dissect Ena/VASP's processive mechanism on bundled filaments. We discovered that Ena's enhanced processivity on trailing barbed ends is specific to fascin bundles, with no enhancement on fimbrin or $\alpha$-actinin bundles. Notably, Ena/VASP's processive run length increases with the number of both fascin-bundled filaments and Ena "arms", revealing avidity facilitates enhanced processivity. Moreover, Ena tetramers form more filopodia than mutant dimer and trimers in \textit{Drosophila} culture cells. Finally, enhanced processivity on trailing barbed ends of fascin-bundled filaments is an evolutionarily conserved property of Ena/VASP homologs human VASP and \textit{C. elegans} UNC-34. These results demonstrate that Ena tetramers are tailored for enhanced processivity on fascin bundles and avidity of multiple arms associating with multiple filaments is critical for this process. Furthermore, we discovered a novel regulatory process whereby bundle size and bundling protein specificity control activities of a processive assembly factor.

\footnotetext{Citation for chapter: Alyssa J. Harker, Harshwardhan H. Katkar, Tamara C. Bidone, Fikret Aydin, Gregory A. Voth, Derek A. Applewhite, David R. Kovar. Ena/VASP processive elongation is modulated by avidity on actin filaments bundled by the filopodia crosslinker fascin. \textit{In revision at Molecular Biology of the Cell.} }

\section{Introduction}\label{ch02-introduction}
Many important cellular functions depend on formation of actin cytoskeleton networks at the correct time and location with specific architectures and dynamics \citep{pollard_actin_2009,campellone_nucleator_2010}. For example, filopodia are filamentous actin (F-actin)-rich finger-like protrusions that elongate from the lamellipodium, a dense, branched F-actin network kept short by capping protein \citep{pollard_cellular_2003} at the cell periphery. Filopodia are important for cell motility and environment sensing. Filopodial actin filaments are assembled by actin elongation factors such as formins and Enabled/vasodilator-stimulated phosphoprotein (Ena/VASP) \citep{mattila_filopodia:_2008}. During filopodia initiation, Ena/VASP localizes to the edge of the lamellipodium, where it competes with capping protein for barbed ends \citep{bear_antagonism_2002, svitkina_mechanism_2003, barzik_ena/vasp_2005, applewhite_ena/vasp_2007, bear_ena/vasp:_2009, winkelman_ena/vasp_2014} and then facilitates generation of long, straight filaments by remaining processively associated with barbed ends and increasing their elongation rate 2- to 7-fold \citep{breitsprecher_clustering_2008,breitsprecher_molecular_2011,pasic_ena/vasp_2008,hansen_vasp_2010,winkelman_ena/vasp_2014,bruhmann_distinct_2017}. The 10-30 filaments in filopodia are bundled primarily by fascin, a globular crosslinking protein containing $\beta$-trefoil domains \citep{vignjevic_role_2006,jansen_mechanism_2011, mellor_role_2010}. Fascin bundles are composed of parallel filaments with narrow spacing, between 8-10 nm \citep{cant_drosophila_1994,edwards_fascins_1995,jansen_mechanism_2011,yang_molecular_2013}. Ena/VASP continues to localize to the tips of mature filopodia, where fascin-bundled filaments ultimately are the same length \citep{faix_making_2006,gupton_filopodia:_2007}, presumably assuring uniform thickness of filopodia required for protrusive force \citep{svitkina_mechanism_2003,winkelman_ena/vasp_2014}.

Ena/VASP is a multidomain homotetramer with homologs in all metazoan cells \citep{sebe-pedros_insights_2013}. A few Ena/VASP homologs have been biochemically characterized, including human VASP \citep{bachmann_evh2_1999, chereau_understanding_2006, breitsprecher_clustering_2008, pasic_ena/vasp_2008, hansen_vasp_2010}, \textit{Drosophila} Enabled \citep{winkelman_ena/vasp_2014}, and \textit{Dictyostelium} VASP \citep{breitsprecher_clustering_2008}. Ena/VASP proteins contain two conserved Ena/VASP homology domains, EVH1 and EVH2 (Figure \ref{fig:ena-bundlers}A). The N-terminus EVH1 domain is important for cellular localization and binds to proteins with FPPPP (FP4) repeats, such as lamellipodin, zyxin, and formin \citep{ball_evh1_2001,bilancia_enabled_2014}. The C-terminus EVH2 domain consists of three smaller subdomains: G-actin binding domain (GAB) \citep{bachmann_evh2_1999,ferron_structural_2007}, F-actin binding domain (FAB) \citep{dominguez_actin_2011}, and a C-terminal coiled-coil tetramerization domain \citep{bachmann_evh2_1999,kuhnel_vasp_2004}. Between the EVH1 and EVH2 domains there is a poly-proline rich region that binds profilin as well as SH3 domains \citep{ferron_structural_2007,hansen_vasp_2010}.

In addition to the leading edge and tips of filopodia, Ena/VASP proteins also localize to focal adhesions and stress fibers \citep{reinhard_46/50_1992,brindle_focal-adhesion_1996}, which are composed of filaments crosslinked by CH domain superfamily crosslinkers, fimbrin/plastin and $\alpha$-actinin. Fimbrin also localizes to the lamellipodia and base of filopodia, and it bundles both parallel and antiparallel filaments with narrow spacing (10-12 nm), similar to fascin \citep{hanein_atomic_1998}. In comparison, $\alpha$-actinin bundles filaments of mixed polarity with much wider spacing (30-36 nm) \citep{sjoblom_alpha-actinin_2008}.

We previously discovered that for F-actin bundles made by human fascin, \textit{Drosophila} Enabled (Ena) remains processively associated with trailing barbed ends (shorter filaments) \mytilde3-fold longer than leading barbed ends (longest filament) (Figure \ref{fig:ena-bundlers}B) \citep{winkelman_ena/vasp_2014}. However, the underlying molecular mechanisms that facilitate Ena's enhanced processivity on bundled filaments remain unclear. We therefore used a combination of in vitro reconstitution with single-molecule multi-color total internal reflection fluorescence microscopy (TIRFM), kinetic modeling, and analysis of \textit{Drosophila} culture cells to characterize the dynamics and function of processive elongation of single and bundled filaments by multiple Ena/VASP homologs including Ena, human VASP, and \textit{C. elegans} UNC-34. Together, our experiments and simulations inform our mechanistic understanding of Ena/VASP on single and bundled filaments, demonstrate that avidity of multiple filaments within fascin bundles and multiple Ena arms leads to increased processivity of tetrameric Ena on trailing barbed ends, and reveal a novel regulatory process whereby the particular F-actin bundling protein matters for Ena/VASP processivity.


\section{Results}\label{ch02-results}

\subsection{Ena is more processive on trailing barbed ends of both human and fly fascin (Singed) bundles}\label{ena-processive-fascin-singed}

To understand what features are important for \textit{Drosophila} Ena's enhanced processivity on trailing barbed ends within human fascin bundles (Figure \ref{fig:ena-bundlers}B) \citep{winkelman_ena/vasp_2014}, we first tested if a different fascin homolog also facilitates enhanced residence times. We used two-color TIRFM to directly visualize the assembly of 1.5 $\mu$M Mg-ATP-actin monomers (15\% Oregon green-labeled) with 15 pM fluorescently labeled SNAP(549)-Ena$\Delta$L (referred to as Ena) (Figure \ref{fig:ena-bundlers}A) and human fascin or fly fascin, Singed. TIRFM allows direct visualization of individual Ena molecule dynamics on single and bundled actin filament barbed ends. Ena's processive run lengths were measured for leading and single filament barbed ends (collectively referred to as leading) as well as trailing barbed ends (Figure \ref{fig:ena-bundlers}C-D). Kaplan-Meier survival curves were calculated from individual Ena processive runs (Figure \ref{fig:ena-bundlers}E-F), revealing that Ena remains associated with trailing barbed ends ($\tau_{fascin}$ = 23.7 s, $\tau_{Singed}$ = 28.1 s) ~3-fold longer than leading barbed ends ($\tau_{fascin}$ = 8.4 s, $\tau_{Singed}$ = 10.1 s) for both human fascin and fly Singed (Figure \ref{fig:ena-bundlers}I, Table \ref{tab:ena-processive}), consistent with our previous findings \citep{winkelman_ena/vasp_2014}. Therefore, enhancement of Ena's processive elongation on trailing barbed ends is not specific to a particular fascin homolog.

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Figure_1_elife.pdf}
\caption[Ena has enhanced processivity on F-actin bundles formed specifically by fascin.]{\textbf{Ena has enhanced processivity on F-actin bundles formed specifically by fascin.} (A) Ena/VASP domain organization and constructs used for Ena, UNC-34, and VASP: Self-labeling tag (SNAP), Ena/VASP homology domain 1 (EVH1), polyproline region (PPR), Ena/VASP homology domain 2 (EVH2) includes G-actin binding domain (G), F-actin binding domain (F), coiled coil region (CC). *Putative domain. Two-color TIRFM visualization of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon green-actin) with 15 pM SNAP(549)-Ena$\Delta$L and unlabeled 130 nM human fascin, 250 nM fly fascin Singed, 125 nM $\alpha$-actinin, or 100 nM fimbrin.  (B) Cartoon Ena/VASPs bound to leading and trailing barbed ends in a fascin bundle. (C and D) Representative experiment of OG-actin with SNAP(549)-Ena$\Delta$L and fascin. Arrows indicate leading (orange), 1st trailing (white), 2nd trailing (yellow) and 3rd trailing (blue) barbed ends. (C) Merged timelapse micrographs. Scale bar, 5 $\mu$m. (D) Merged kymograph of filament length (scale bar, 5 $\mu$m) over time (time bar, 10 s).}
\label{fig:ena-bundlers}
\end{figure}

% Continues caption on next page. Requires package ccaption.
\begin{figure}
  \contcaption{(continued) (E-H) Kaplan-Meier curves representing average processive run lengths ($\tau$) for Ena with (E) fascin, (F) Singed, (G) $\alpha$-actinin, or (H) fimbrin on leading (red) and trailing (blue) barbed ends. Error bars, 95\% CI. n $\geq$ 127. (I) Average processive run lengths for leading (red) and trailing (blue) barbed ends shown in E-H for 2-filament bundles with fascin, Singed, $\alpha$-actinin, or fimbrin. P values (*$<$0.0001). Error bars, 95\% CI. (J) Average processive run lengths for antiparallel and parallel 2-filament $\alpha$-actinin (green) or fimbrin (purple) bundles. Error bars, 95\% CI. n $\geq$ 64. (K) Fold increase of barbed end elongation rates of Ena on fascin (yellow), Singed (blue), $\alpha$-actinin (green), or fimbrin (purple) bundled filaments. Error bars, SEM. n $\geq$ 5 barbed ends from at least 2 movies.}
\end{figure}


\subsection{Ena's residence time is not enhanced on trailing barbed ends of fimbrin and \texorpdfstring{$\alpha$}{a}-actinin bundles}\label{ena-processive-fimbrin-aact}

To determine if diverse bundle architectures are similarly sufficient to enhance Ena's processivity on trailing barbed ends, we tested the effect of bundling proteins with distinct properties (fimbrin and $\alpha$-actinin, see introduction). First, we measured elongation rates of Ena-bound leading and trailing barbed ends of filaments bundled by human fascin, fly fascin Singed, $\alpha$-actinin, or fimbrin. Two-color TIRFM visualization of control and Ena-bound barbed ends revealed a similar fold increase in Ena-mediated actin elongation for leading (\mytilde2.2- to 3-fold) and trailing (\mytilde2- to 2.5-fold) barbed ends with all four bundling proteins (Figures \ref{fig:ena-bundlers}K, \ref{fig:bulk-elongation}, Tables \ref{tab:ena-elongation}, \ref{tab:ena-p-elongation-bundlers}). Therefore, Ena's barbed end elongation enhancement is bundling protein independent. 

\begin{table}[hbtp]
\centering 
\begin{tabulary}{0.7\textwidth}{C C C C C}
\toprule Ena/VASP & Bundling Protein & Leading\textsuperscript{a} (s) & Trailing\textsuperscript{a} (s) & L/T p-value\textsuperscript{b} \\ 
\midrule 
Ena Tetramer & Fascin & 8.4\hspace{0.5cm} [7.7,9.1] (254) & 23.7\hspace{0.3cm} [22.0,25.8]\hspace{0.3cm} (511) & ${<0.0001}$ \\ 
Ena Tetramer & Singed & 10.1 [9.4,11.0] (184) & 28.1\hspace{0.3cm} [27.0,29.4]\hspace{0.3cm} (328) & ${<0.0001}$ \\ 
Ena Tetramer & $\alpha$-actinin & 10.2 [9.5,11.1] (284) & 9.4\hspace{0.7cm} [8.4,10.5]\hspace{0.3cm} (176) & 0.64 \\
Ena Tetramer & Fimbrin & 9.0\hspace{0.3cm} [8.2,10.0] (127) & 8.7\hspace{0.8cm} [7.7,10.0]\hspace{0.3cm} (183) & 0.91 \\
VASP Tetramer & Fascin & 1.2\hspace{0.5cm} [0.9,1.6] (213) & 3.3\hspace{0.9cm} [3.1,3.5]\hspace{0.5cm} (348) & ${<0.0001}$ \\
VASP Tetramer & Singed & 1.0\hspace{0.5cm} [0.7,1.5] (187) & 3.5\hspace{0.9cm} [3.2,3.8]\hspace{0.5cm} (463) & ${<0.0001}$ \\
UNC-34 Tetramer & Fascin & 1.7\hspace{0.4cm} [1.4,2.2]\hspace{0.3cm} (82) & 3.7\hspace{0.9cm} [3.4,4.0]\hspace{0.3cm} (266) & ${<0.0001}$ \\
Ena\hspace{0.2cm} Trimer & Fascin & 6.2\hspace{0.5cm} [5.6,6.9] (322) & 9.8\hspace{0.8cm} [9.2,10.4]\hspace{0.3cm} (299) & ${<0.0001}$ \\
Ena\hspace{0.3cm} Dimer & Fascin & 1.3\hspace{0.5cm} [1.0,1.7] (376) & 1.8\hspace{1.0cm} [1.5,2.4]\hspace{0.3cm} (418) & 0.01 \\
\bottomrule
\end{tabulary}
\caption[Comparison of Ena/VASP proteins' residence time on leading and trailing barbed ends.]{\textbf{Comparison of Ena/VASP proteins' residence time on leading and trailing barbed ends.} \\
   \textsuperscript{a} Values of average processive lifetime (s) [95\%CI] (n) where n is the number of Ena/VASP binding events measured in at least three movies for Leading or Trailing barbed ends.\\
   \textsuperscript{b} Log Rank p-value comparing Leading and Trailing average processive lifetime. \\}
\label{tab:ena-processive}
\end{table}

\begin{figure}
\centering
\includegraphics[width=14cm]{img/ch02/bulk_elongation_thesis.pdf}
\caption[Bulk elongation shows average elongation rates of Ena-bound and control filaments.]{\textbf{Bulk elongation show average elongation rates of Ena-bound and control filaments.} Example scatter plots fit with a linear equation for the length of control filaments or Ena-bound filaments over time in the presence of 15 pM SNAP(549)-Ena$\Delta$L with either (A) 130 nM human fascin, (B) 250 nM fly fascin Singed, (C) 125 nM $\alpha$-actinin, or (D) 100 nM fimbrin. A linear fit gives the bulk elongation rate for a single movie. Average elongation rates are given in Table \ref{tab:ena-elongation}.}
\label{fig:bulk-elongation}
\end{figure}

Conversely, Ena's enhanced processivity on trailing barbed ends is specific to fascin bundles. The average processive run length on leading barbed ends with all four bundling proteins is similar, \mytilde10 sec (Figure \ref{fig:ena-bundlers}I). However, there is no enhancement of Ena's average residence time on trailing barbed ends of $\alpha$-actinin ($\tau$ = 9.4 s) or fimbrin ($\tau$ = 8.7 s) bundles (Figure \ref{fig:ena-bundlers}G-I, Table \ref{tab:ena-processive}). Therefore, F-actin bundling proteins are not universally sufficient to enhance Ena's processivity on trailing barbed ends. Although fascin exclusively forms parallel bundles, $\alpha$-actinin and fimbrin form bundles composed of filaments with mixed polarities. We therefore compared Ena's residence time on trailing barbed ends in parallel and antiparallel two-filament bundles. For both fimbrin and $\alpha$-actinin bundles, the average residence time for trailing parallel and antiparallel barbed ends is equivalent; thus, neither bundler enhances Ena's processivity (Figures \ref{fig:ena-bundlers}J, Table \ref{tab:ena-parallel}). Fimbrin can bind to single filaments \citep{skau_actin_2011}, so to control for potential hindrance of Ena/VASP association with F-actin by fimbrin binding to single filaments we tested a low concentration of fimbrin that could still bundle (Figure \ref{fig:ena-low-fim}). We found that even with low concentrations of fimbrin that there was no enhancement of Ena processivity. Therefore, neither "fascin-like" filament spacing (8-10 nm) nor polarity (parallel) of actin filaments within bundles is sufficient to facilitate increased processivity on trailing barbed ends. Given that Ena's \mytilde3-fold enhancement of processivity on trailing barbed ends is specific to fascin, different bundling proteins could regulate Ena's specific activity for different F-actin networks. 

\begin{table}[hbtp]
\centering 
\begin{tabulary}{1.0\textwidth}{C C C C C C C C C}
\toprule Ena\slash VASP & Bundling ${\textup{Protein}}$ & Bound Leading\textsuperscript{a} (sub/s) & Bound Trailing\textsuperscript{a} (sub/s) & Control Leading\textsuperscript{b} (sub/s) &  Control Trailing\textsuperscript{b} (sub/s)  &  Fold Change Leading\textsuperscript{c} & Fold Change Trailing\textsuperscript{c} & n\textsuperscript{d} \\ \midrule 
Ena Tetramer & Fascin & 25.6 $\pm$0.8 & 16.8 $\pm$1.1 & 10.0\hspace{0.1cm} $\pm$1.0 & 7.9\hspace{0.3cm} $\pm$0.2 & 2.56 $\pm$0.08 & 2.1\hspace{0.1cm} $\pm$0.2 & 2 \\ 
Ena Tetramer & Singed & 22.4 $\pm$0.6 & 20.0 $\pm$0.1 & 10.0\hspace{0.1cm} $\pm$0.9 & 10.0\hspace{0.1cm} $\pm$0.2 & 2.24 $\pm$0.06 & 2.00 $\pm$0.02 & 2 \\ 
Ena Tetramer & ${\alpha\mhyphen\textup{actinin}}$ & 27.6 $\pm$2.7 & 25.43 $\pm$0.03 & 10.0\hspace{0.1cm} $\pm$0.2 & 11.5\hspace{0.1cm} $\pm$0.5 & 2.8\hspace{0.1cm} $\pm$0.3 & 2.2\hspace{0.1cm} $\pm$0.1 & 2 \\
Ena Tetramer & ${\textup{Fimbrin}}$ & 29.9 $\pm$3.7 & 21.9 $\pm$1.0 & 10.0\hspace{0.1cm} $\pm$0.1 & 9.0\hspace{0.3cm} $\pm$0.3 & 3.0\hspace{0.1cm} $\pm$0.4 & 2.43 $\pm$0.04 & 2 \\
VASP Tetramer & Fascin & 23.6 $\pm$3.9 & 18.8 $\pm$4.6 & 10.0\hspace{0.1cm} $\pm$1.4 & 8.0\hspace{0.3cm} $\pm$0.5 & 2.4\hspace{0.1cm} $\pm$0.4 & 2.4\hspace{0.1cm} $\pm$0.7 & 2 \\
${\textup{UNC}\mhyphen34}$ Tetramer & Fascin & 27.2 $\pm$2.3 & 20.2 $\pm$3.4 & 10.0\hspace{0.1cm} $\pm$1.8 & 8.0\hspace{0.3cm} $\pm$1.5 & 2.7\hspace{0.1cm} $\pm$0.2 & 2.7\hspace{0.1cm} $\pm$0.9 & 2 \\
Ena Trimer & Fascin & 17.4 $\pm$1.3 & 14.1 $\pm$0.8 & 10.0\hspace{0.1cm} $\pm$0.2 & 8.70 $\pm$0.05 & 1.7\hspace{0.1cm} $\pm$0.1 & 1.6\hspace{0.1cm} $\pm$0.1 & 2 \\
Ena Dimer & Fascin & 14.5 $\pm$0.2 & 14.1 $\pm$2.2 & 10.0\hspace{0.1cm} $\pm$0.2 & 9.8\hspace{0.3cm} $\pm$0.9 & 1.45 $\pm$0.02 & 1.5\hspace{0.1cm} $\pm$0.4 & 2 \\
\bottomrule
\end{tabulary}
\caption[Comparison of actin elongation rates with and without (control) Ena/VASP bound.]{\textbf{Comparison of actin elongation rates with and without (control) Ena/VASP bound.} \\
   \textsuperscript{a} Normalized actin elongation rate (sub/s) of Ena/VASP bound Leading or Trailing barbed ends to Control Leading.\\
   \textsuperscript{b} Normalized actin elongation rate (sub/s) of Ena/VASP free Leading or Trailing barbed ends to Control Leading.\\
   \textsuperscript{c} Fold change in actin elongation rate of Ena/VASP bound over Ena/VASP free Leading or Trailing barbed ends.\\
   \textsuperscript{d} n is the number of movies analyzed. Each movie had at least five filaments with at least 50 length measurements for each movie.}
\label{tab:ena-elongation}
\end{table}

\begin{table}[!htb]
\centering
\begin{tabular}{ c c c c c }
\toprule 
Leading\slash Trailing\textsuperscript{a} & Fascin & Singed & $\alpha$-actinin & Fimbrin \\
\midrule
Fascin & 1 / 1 & 0.3 / 1 & 0.6 / 0.7 & 0.4 / 0.3 \\
Singed & 0.3 / 1 & 1 / 1 & 0.4 / 0.7 & 0.3 / 0.3 \\
$\alpha$-actinin & 0.6 / 0.7 & 0.4 / 0.7 & 1 / 1 & 0.7 / 0.3 \\
Fimbrin & 0.4 / 0.3 & 0.3 / 0.3 & 0.7 / 0.3 & 1 / 1 \\
\bottomrule
\end{tabular}
\caption[p-values for comparisons of fold change in actin elongation rate with Ena on different bundling proteins for both leading and trailing filaments.]{\textbf{p-values for comparisons of fold change in actin elongation rate with Ena on different bundling proteins for both leading and trailing filaments.} \\
\textsuperscript{a} p-values from student's two-tailed t-test with unequal variance between fold change of actin elongation rates when Ena is bound to the Leading or Trailing barbed end.}
\label{tab:ena-p-elongation-bundlers}
\end{table}

\begin{figure}[ht]
\centering
\includegraphics[width=12cm]{img/ch02/low_fimbrin.pdf}
\caption[Low concentrations of fimbrin also do not enhance Ena processivity.]{\textbf{Low concentrations of fimbrin also do not enhance Ena processivity.} Kaplan-Meier curves representing average processive run lengths ($\tau$) for Ena with (A-B) 100 nM fimbrin or (C-D) 10 nM fimbrin on (A,C) leading (red) and trailing (blue) barbed ends or further separated into (B,D) single filaments (red), or bundles with 2 (blue) and $\geq$3 (green) filaments. Error bars, 95\% CI. n $\geq$ 58.  Kaplan-Meier curve in (A) also shown in Figure \ref{fig:ena-bundlers}H.}
\label{fig:ena-low-fim}
\end{figure}

\begin{table}
\centering
\begin{tabular}{ c c c c c }
\toprule 
Ena\slash VASP & Bundling Protein & Parallel\textsuperscript{a} (s) & Antiparallel\textsuperscript{a} (s) & A/P p-value\textsuperscript{b} \\
\midrule
Ena Tetramer & Fascin & 16.8 [14.3,19.7] (201) & N/A & N/A \\
Ena Tetramer & Singed & 21.7 [20.4,23.3] (155) & N/A & N/A \\
Ena Tetramer & $\alpha$-actinin & 9.7 [8.1,12.0] (77) & 8.8 [8.0,9.8] (90) & 0.52 \\
Ena Tetramer & Fimbrin & 8.9 [7.7,10.4] (64) & 6.9 [5.7,8.7] (106) & 0.53 \\
\bottomrule
\end{tabular}
\caption[Comparison of Ena/VASP proteins' residence time on parallel and antiparallel bundled F-actin.]{\textbf{Comparison of Ena/VASP proteins' residence time on parallel and antiparallel bundled F-actin.} \\
\textsuperscript{a} Values of average processive lifetime (s) [95\%CI] (n) where n is the number of Ena/VASP binding events measured in at least three movies for parallel or antiparallel barbed ends on 2-filament bundles. Fascin results are equal to the values in 2 filaments because fascin only makes parallel bundles. \\
\textsuperscript{b} Log Rank p-value comparing average processive lifetime trailing barbed ends in parallel and antiparallel bundles.
}
\label{tab:ena-parallel}
\end{table}

\subsection{Ena's processive run length increases with bundle size}\label{ena-bundle-size}

Filopodia are composed of \mytilde10-30 actin filaments bundled by fascin \citep{svitkina_mechanism_2003,faix_making_2006}, suggesting an avidity mechanism where enhanced processivity depends on Ena simultaneously associating with a barbed end and sides of neighboring filaments. To test whether the number of filaments in a fascin bundle positively correlates with processive run length, we determined the dependence of Ena's enhanced processivity on fascin bundle size (Figure \ref{fig:ena-filaments}A). Average run lengths on trailing barbed ends (Figure \ref{fig:ena-bundlers}E-F) was thereby parsed into 2-filament bundles or 3- or more filament bundles for both human and fly fascin (Figure \ref{fig:ena-filaments}B-D, Table \ref{tab:ena-processive}). Ena's average residence time on trailing barbed ends of a 2-filament bundle ($\tau_{fascin}$ = 16.8 s, $\tau_{Singed}$ = 21.7 s) is \mytilde2-fold longer than on single filament barbed ends ($\tau_{fascin}$ = 8.9 s, $\tau_{Singed}$ = 10.0 s). Furthermore, there is an additional \mytilde1.5-fold increase in processivity when Ena is bound to trailing barbed ends of 3- or more filament bundles ($\tau_{fascin}$ = 26.0 s, $\tau_{Singed}$ = 32.2 s) (Figure \ref{fig:ena-filaments}D). Therefore, consistent with an avidity effect, Ena's processivity increases with the number of fascin-bundled filaments. In contrast, when comparing Ena's residence time on single filaments compared to the trailing barbed end of a 2-filament bundle, there is no difference with respect to fascin concentration (Figure \ref{fig:fascin-range}, Table \ref{tab:ena-p-fascin-range}). This suggests that changing the fascin concentration in solution does not affect the property of fascin that causes a specific enhancement of Ena's processivity. 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Figure_2_elife.pdf}
\caption[Ena/VASP's processive run length increases with the number of filaments in a fascin bundle.]{\textbf{Ena/VASP's processive run length increases with the number of filaments in a fascin bundle.} (A) Cartoons of Ena/VASP on a single filament and 2- and 3-filament fascin bundles. (B-G) Two-color TIRFM visualization of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon green-actin) with fly SNAP(549)-Ena$\Delta$L (red), human SNAP(549)-VASP or worm SNAP(549)-UNC-34 and unlabeled 130 nM human fascin or 250 nM Singed as indicated. (B and C) Kaplan-Meier curves representing average processive run lengths ($\tau$) for 15 pM Ena with (B) fascin or (C) Singed on single filaments (red), or bundles with 2 (blue) and $\geq$3 (green)  filaments. Error bars, 95\% CI. n $\geq$ 98. (D) Average processive run lengths for increasing number of filaments in fascin (yellow) or Singed (blue) bundles shown in B and C. Error bars, 95\% CI. P values (*$<$0.0001). (E and F) Kaplan-Meier curves representing run lengths ($\tau$) for (E) 25 pM VASP or (F) 18 pM UNC-34 with fascin on single filaments (red), or bundles with 2 (blue) and $\geq$3 (green) filaments. Error bars, 95\% CI. n $\geq$ 60. (G) VASP and UNC-34 average processive run lengths for increasing number of filaments in fascin bundles shown in E and F. Error bars, 95\% CI. P values (*$<$0.0001, **0.002).}
\label{fig:ena-filaments}
\end{figure}

\begin{table}[hbtp]
\footnotesize
\centering 
\begin{tabulary}{.9\textwidth}{C C C C C C C C}
\toprule 
Ena/VASP & Bundling Protein & 1 fil.\textsuperscript{a} (s) &  2 fil.\textsuperscript{a} (s) & $\geq$3 fil.\textsuperscript{c} (s) & 1/2/$\geq$3 p-value\textsuperscript{b} & 1/2 p-value\textsuperscript{b} & 2/3 p-value\textsuperscript{b} \\ 
\midrule 
Ena Tetramer & Fascin & 8.9 [7.5,10.6] (107) & 16.8 [14.3,19.7] (201) & 26.0 [23.7,28.6] (308) & ${<0.0001}$ & ${<0.0001}$ & ${<0.0001}$ \\ 
Ena Tetramer & Singed & 10.0 [9.2,10.8] (98) & 21.7 [20.4,23.3] (155) & 32.3 [31.0,33.6] (176) & ${<0.0001}$ & ${<0.0001}$ & ${<0.0001}$ \\ 
Ena Tetramer & $\alpha$-actinin & 9.1 [8.3,10.2] (165) & 8.9 [7.7,10.6] (116) & 8.7\hspace{0.1cm} [7.8,9.9] (60) & 0.6 & 0.8 & 0.9 \\
Ena Tetramer & Fimbrin & 8.2\hspace{0.8cm} [7.1,9.8] (63) & 7.6\hspace{0.1cm} [6.6,9.1] (121) & 10.2 [8.6,12.6] (64) & 0.3 & 0.2 & 0.1\\
VASP Tetramer & Fascin & 1.0\hspace{0.8cm} [0.7,1.8] (123) & 2.6\hspace{0.1cm} [2.4,2.9] (207) & 4.2\hspace{0.1cm} [3.9,4.4] (143) & ${<0.0001}$ & ${<0.0001}$ & ${<0.0001}$\\
VASP Tetramer & Singed & 0.9\hspace{0.8cm} [0.6, 1.6] (118) & 2.8\hspace{0.1cm} [2.6,3.2] (224) & 4.1\hspace{0.1cm} [3.7,4.7] (220) & ${<0.0001}$ & ${<0.0001}$ & ${<0.0001}$ \\
UNC-34 Tetramer & Fascin & 1.2\hspace{0.8cm} [0.9,1.8] (65) & 2.9\hspace{0.1cm} [2.7,3.3] (123) & 3.9\hspace{0.1cm} [3.5,4.5] (144) & ${<0.0001}$ & ${<0.0001}$ & 0.002 \\
Ena Trimer & Fascin & 5.3\hspace{0.8cm} [4.7,6.1] (151) & 8.9\hspace{0.1cm} [8.3,9.6] (206) & 11.2 [10.2,12.6] (93) & ${<0.0001}$ & ${<0.0001}$ & 0.0008\\
Ena Dimer & Fascin & 1.2\hspace{0.8cm} [0.9,1.9] (197) & 1.5\hspace{0.1cm} [1.2,2.0] (261) & 2.5\hspace{0.1cm} [1.9,3.8] (122) & 0.03 & 0.3 & 0.03 \\
\bottomrule
\end{tabulary}
\caption[Comparison of Ena/VASP proteins' residence time on various bundled F-actin.]{\textbf{Comparison of Ena/VASP proteins' residence time on various bundled F-actin.} \\
   \textsuperscript{a} Values of average processive lifetime (s) [95\%CI] (n) where n is the number of Ena/VASP binding events measured in at least three movies for 1 filament (fil.), 2 filaments, or greater than or equal to 3 filaments barbed ends.\\
   \textsuperscript{b} Log Rank p-value comparing 1 filament, 2 filaments, or greater than or equal to 3 filaments average processive lifetime.}
\label{tab:ena-processive-filaments}
\end{table}

\begin{figure}
\centering
\includegraphics[width=13cm]{img/ch02/Supp_Figure_1_mboc2.pdf}
\caption[Ena's enhanced processivity is consistent over a range of fascin.]{\textbf{Ena's enhanced processivity is consistent over a range of fascin.} (A-C) Kaplan-Meier curves representing average processive run lengths ($\tau$) for 15 pM Ena with (A) 50 nM (B) 130 nM or (C) 500 nM fascin on single filaments (red), or 2-filament bundles (blue). Error bars, 95\% CI. n $\geq$ 77. (D) Average processive run lengths on single filaments (red) and 2-filament bundles (blue) with increasing concentration of fascin. Error bars, 95\% CI. P values (*$<$0.0001, **0.0002). 130 nM fascin data in (B) and (D) was also reported in \ref{fig:ena-filaments}B and D respectively.}
\label{fig:fascin-range}
\end{figure}

\begin{table}[!htb]
\centering
\begin{tabular}{c c c c c}
\toprule 
Bundling protein & 1 fil.\textsuperscript{a} (s) & 2 fil.\textsuperscript{a} (s) & 1/2 p-value\textsuperscript{b} & Fold change\textsuperscript{c} \\
\midrule
50 nM Fascin & 8.1 [6.5,10.2] (142) & 16.5 [14.2,19.1] (80) & 0.0002 & 2.3 \\
130 nM Fascin\textsuperscript{d} & 8.9 [7.5,10.6] (107) & 16.8 [14.3,19.1] (201) & $<0.0001$ & 1.9 \\
500 nM Fascin & 6.1 [5.0,7.5] (136) & 16.2 [14.7,18.0] (77) & $<0.0001$ & 3.0 \\
\bottomrule
\end{tabular}
\caption[Average processive lifetime and p-values for Ena tetramers with different fascin concentrations.]{\textbf{Average processive lifetime and p-values for Ena tetramers with different fascin concentrations.} \\
   \textsuperscript{a} Values of average Ena\textsubscript{Tetramer} processive lifetime on Leading or 2-filament barbed ends (s) [95\%CI] (n) where n is the number of Ena/VASP binding events measured in at least two movies for barbed ends of leading filaments or 2-filament bundles.\\
   \textsuperscript{b} Log Rank p-value comparing average processive lifetime of barbed ends of leading filaments and 2-filament bundles. \\
   \textsuperscript{c} Fold change in average processive lifetime between leading and 2-filament bundle barbed ends. \\
   \textsuperscript{d} Data also shown in Table \ref{tab:ena-processive-filaments}.\\
   }
\label{tab:ena-p-fascin-range}
\end{table}

\subsection{Human VASP and worm UNC-34 also have enhanced processive properties on fascin bundles}\label{vasp-unc34-processive}

To determine whether enhanced processivity on fascin-bundled trailing filament barbed ends is conserved among Ena/VASP family members, we extended our analysis to human VASP and worm UNC-34 (Figure \ref{fig:ena-bundlers}A). Human VASP is a well-characterized Ena/VASP protein \citep{bachmann_evh2_1999,chereau_understanding_2006,breitsprecher_clustering_2008,pasic_ena/vasp_2008,hansen_vasp_2010}, whereas UNC-34 had not yet been biochemically characterized in vitro despite multiple in vivo studies \citep{sheffield_c._2007,fleming_role_2010,havrylenko_wave_2015}. 

For our initial characterization of the three homologs, we measured the affinity for barbed ends and effect on actin elongation for Ena, VASP, and UNC-34. Initially, the effect of Ena/VASP homologs on actin elongation rates and their apparent affinity (Kd, app) for barbed ends was determined by single-color TIRFM visualization of spontaneous assembly of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon Green) over a range of concentrations for each unlabeled Ena/VASP homolog (Figure \ref{fig:ena-homologs}A-F). All three Ena/VASP homologs increase actin elongation by a similar amount, \mytilde1.6- to \mytilde2.7-fold, at or near saturating conditions but have somewhat varying affinities for actin filament barbed ends ranging from 3.2 nM (Ena) to 6.7 nM (UNC-34) to 12.2 nM (VASP) (Figure \ref{fig:ena-homologs}F). Likewise, bulk seeded pyrene actin assembly assays also show that all three Ena/VASP homologs increase actin elongation rates by similar amounts, and fits of assembly rate over a range of Ena/VASP concentrations revealed apparent affinities for barbed ends ranging from 0.7 nM (Ena) to 10.2 nM (UNC-34) to 10.8 nM (VASP) (Figure \ref{fig:ena-homologs}G-H). We then used two-color TIRFM visualizations of red-labeled Ena, VASP, and UNC-34 on fascin bundles to measure actin elongation rates of Ena/VASP-bound leading and trailing barbed ends (Figure \ref{fig:ena-homologs}I). All three Ena/VASP homologs similarly increase actin elongation \mytilde2- to 3-fold on both leading and trailing barbed ends (Figure \ref{fig:ena-homologs}J, Tables \ref{tab:ena-p-elongation-bundlers}, \ref{tab:ena-p-elongation-homologs}). Enhancement of actin elongation rates by Ena and VASP are similar to previously reported values \citep{hansen_vasp_2010,winkelman_ena/vasp_2014,bruhmann_distinct_2017} and the actin elongation properties of UNC-34 are in good agreement with the other homologs. Therefore, though Ena, VASP, and UNC-34 vary in their barbed end affinity, they all similarly increase the actin elongation rate of both leading and trailing barbed ends of fascin-bundled filaments.

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Supp_Figure_1.pdf}
\caption[Ena/VASP homologs have generally conserved processive actin elongation properties.]{\textbf{Ena/VASP homologs have generally conserved processive actin elongation properties.} (A-F) Single-color TIRFM of the spontaneous assembly of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon Green) with worm UNC-34, fly Ena, and human VASP. (A and B) Timelapse micrographs (scale bar, 5 $\mu$m), and (C and D) corresponding kymographs (scale bar, 1 $\mu$m; time bar, 5 s) for (A and C) actin alone or with (B and D) 18 nM UNC-34. Yellow arrowheads indicate barbed ends. (E) Length of individual filaments over time for actin only (black) and UNC-34 (red). (F) Fold increase in elongation rate over increasing concentration of Ena (green), VASP (blue), and UNC-34 (red). Curve fits revealed the indicated apparent dissociation constants (Kd, app) of Ena/VASP for the barbed end. $n \geq 5$ filaments for at least 2 movies. (G and H) Seeded assembly: addition of 0.5 $\mu$M Mg-ATP-actin monomers (20\% pyrene-labeled) to the barbed end of 0.5 $\mu$M preassembled actin filaments. (G) Time course of seeded assembly alone (black and gray) or with a range of Ena concentrations. (H) Dependence of the initial barbed end assembly rate on Ena/VASP concentration. Curve fits revealed the indicated apparent dissociation constants (Kd, app) of Ena/VASP for the barbed end. Error, SEM. $n \geq 3$. (I and J) Two-color TIRFM visualization of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon Green) with 25 pM SNAP(549)-VASP, 18 pM SNAP(549)-UNC-34 or 15 pM SNAP(549)-Ena, and unlabeled 130 nM human fascin. (I) Kymograph of leading and trailing barbed ends of a fascin bundle with SNAP(549)-VASP (red) and OG actin (green). Dashed blue (trailing) and yellow (leading) lines indicate bound VASP. Scale bar, 5 $\mu$m. Time bar, 10 s. (J) Average elongation rate of leading and trailing filament barbed ends on fascin bundles with actin alone, Ena, VASP, or UNC-34. Error bars, SEM. $n \geq 5$ filaments for at least 2 movies. Elongation rate for Ena in (J) also shown in Figure \ref{fig:ena-bundlers}K.}
\label{fig:ena-homologs}
\end{figure}

\begin{table}[!htb]
\centering
\begin{tabular}{ c c c c c c }
\toprule 
Leading\slash Trailing\textsuperscript{a} & Ena\textsubscript{Tetramer} & VASP & UNC-34 & Ena\textsubscript{Trimer} & Ena\textsubscript{Dimer} \\
\midrule
Ena\textsubscript{Tetramer} & 1 / 1 & 0.7 / 0.8 & 0.6 / 0.6 & \underline{0.05} / 0.2 & \underline{0.03} / 0.3 \\
VASP & 0.7 / 0.8 & 1 / 1 & 0.5 / 0.8 & 0.3 / 0.5 & 0.3 / 0.4 \\
UNC-34 & 0.6 / 0.6 & 0.5 / 0.8 & 1 / 1 & 0.09 / 0.4 & 0.1 / 0.4 \\
Ena\textsubscript{Trimer} & \underline{0.05} / 0.2 & 0.3 / 0.5 & 0.09 / 0.4 & 1 / 1 & 0.3 / 0.8 \\
Ena\textsubscript{Dimer} & \underline{0.03} / 0.3 & 0.3 / 0.4 & 0.1 / 0.4 & 0.3 / 0.8 & 1 / 1 \\
\bottomrule
\end{tabular}
\caption[p-values for comparisons of fold change in actin elongation rate with different Ena/VASPs on both leading and trailing filaments of fascin bundles.]{\textbf{p-values for comparisons of fold change in actin elongation rate with different Ena/VASPs on both leading and trailing filaments of fascin bundles.} \\
\textsuperscript{a} p-values from student's two-tailed t-test with unequal variance between fold change of actin elongation rates when Ena/VASP is bound to the leading or trailing barbed end. Underlining shows p-values $\leq0.05$.}
\label{tab:ena-p-elongation-homologs}
\end{table}

To test if different Ena/VASP homologs have similarly enhanced processive properties on fascin bundles, two-color TIRFM visualization of 1.5 $\mu$M Mg-ATP-actin (15\% Oregon Green) was used to quantify the processive run lengths of fluorescently labeled VASP and UNC-34 on fascin bundles (Figure \ref{fig:ena-filaments}E-G). The average residence time of both VASP (1.0 s) and UNC-34 (1.2 s) on single filament barbed ends is \mytilde9-fold shorter than Ena (8.9 s), as expected from lower apparent affinities for barbed ends and previously reported values \citep{hansen_vasp_2010}. Yet, like Ena, both VASP and UNC-34 have \mytilde2.5-fold longer processive run lengths on trailing barbed ends of 2-filament bundles ($\tau_{VASP}$ = 2.6 s, $\tau_{UNC-34}$ = 2.9 s), with an additional \mytilde1.5-fold increase on trailing barbed ends of 3- or more filament bundles ($\tau_{VASP}$ = 4.2 s, $\tau_{UNC-34}$ = 3.9 s) (Figure \ref{fig:ena-filaments}E–G, Table \ref{tab:ena-processive}). Therefore, enhanced processivity on fascin-bundled trailing barbed ends is conserved from worms to flies to humans, suggesting that enhanced processivity is important for Ena/VASP's activity in cells. 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Figure_3_elife.pdf}
\caption[Ena's processive run length increases with the number of Ena ‘arms'.]{\textbf{Ena's processive run length increases with the number of Ena ‘arms'.} (A) Cartoon and domain organizations of Ena\textsubscript{Tetramer}, Ena\textsubscript{Trimer}, and Ena\textsubscript{Dimer}. (B-E) Two-color TIRFM visualization of 1.5 $\mu$M Mg-ATP-actin (15\% Alexa 488-actin) with indicated SNAP(549)-Ena construct and 130 nM fascin. (B) Fold increase of barbed end elongation rates of Ena\textsubscript{Dimer} (orange), Ena\textsubscript{Trimer} (purple), and Ena\textsubscript{Tetramer} (yellow). Error bars, SEM. n $\geq$ 5 barbed ends from at least 2 movies. P values (*$\leq$0.05) (C and D) Kaplan-Meier curves representing average processive run lengths ($\tau$) for (C) 50 pM MBP-SNAP(549)-Ena$\Delta$L$\Delta$CC-GCN4 or (D) 70 pM MBP-SNAP(549)-Ena$\Delta$L$\Delta$CC-Foldon with fascin on single filaments (red) or bundles with 2 (blue) and $\geq$3 (green) filaments. Error bars, 95\% CI. n $\geq$ 93. (E) Average processive run length for increasing number of Ena "arms" on single filaments (red), or fascin bundles with 2 (blue) and $\geq$3 (green) filaments shown in C and D. Error bars, 95\% CI. P values (*$<$0.0001, **0.0008, ***0.03). Ena\textsubscript{Tetramer} data in (B) and (E) was also reported in \ref{fig:ena-bundlers}K and \ref{fig:ena-filaments}D respectively. }
\label{fig:ena-arms}
\end{figure}

\subsection{Enhanced elongation and processive run length increases with the number of Ena arms}\label{ena-processive-arms}

Wildtype Ena is a tetrameric protein \citep{kuhnel_vasp_2004, winkelman_ena/vasp_2014}, with four arms that could facilitate simultaneous association with a barbed end, neighboring actin filaments, and/or actin monomers for processive elongation. Since we observed that Ena's average processive run length increases with number of fascin-bundled filaments (Figure \ref{fig:ena-filaments}), we investigated the importance of Ena's oligomeric state by measuring actin elongation and processive properties of dimeric and trimeric Ena. Dimer and trimer constructs were formed by replacing Ena's coiled-coil tetramerization domain with a GCN4 dimerization domain \citep{harbury_switch_1993} or a Foldon trimerization domain (Figure \ref{fig:ena-arms}A) \citep{guthe_very_2004, papanikolopoulou_formation_2004}; and the oligomeric state was verified by gel filtration and multi-angle light scattering (Figure \ref{fig:ena-sizes}A-C). Two-color TIRFM was used to visualize 1.5 $\mu$M Mg-ATP-actin (15\% Alexa-488 labeled) with SNAP(549)-Ena$\Delta$L$\Delta$CC-GCN4 (referred to as Ena\textsubscript{Dimer}) or SNAP(549)-Ena$\Delta$L$\Delta$CC-Foldon (referred to as Ena\textsubscript{Trimer}) on fascin bundles. First, we measured actin elongation rates of Ena-bound leading and trailing barbed ends (Figure \ref{fig:ena-arms}B, Tables \ref{tab:ena-elongation}, \ref{tab:ena-p-elongation-homologs}). While all constructs increase actin's elongation rate on both leading and trailing filaments, the fold increase is positively correlated with the number of Ena arms. Ena\textsubscript{Tetramer} has the largest enhancement of actin elongation (2.56-fold leading, 2.1-fold trailing), followed by Ena\textsubscript{Trimer} (1.74-fold leading, 1.62-fold trailing), and then Ena\textsubscript{Dimer} (1.45-fold leading, 1.46-fold trailing). 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Supp_Figure_2_PNAS.pdf}
\caption[Ena constructs have the predicted oligomerization state.]{\textbf{Ena constructs have the predicted oligomerization state.} (A) UV traces for Ena\textsubscript{Dimer} (orange), Ena\textsubscript{Trimer} (purple), and Ena\textsubscript{Tetramer} (black) from size exclusion gel filtration using a Sepharose 6 Increase column. Peaks are labeled and fractions collected are shaded. (B) The 12.5\% SDS-PAGE of fractions from A. Fractions showing each construct are boxed. (C) Size exclusion chromatography followed by multi-angle light scattering (SEC-MALS) was used to determine the relative size of Ena\textsubscript{Dimer} (orange), Ena\textsubscript{Trimer} (purple), and Ena\textsubscript{Tetramer} (black). Dependence of (D) Ena\textsubscript{Dimer}, (E) Ena\textsubscript{Trimer}, or (F) Ena\textsubscript{Tetramer} processive run length on its respective fluorescence intensity for an individual movie. Linear correlation fit shown with 95\% CI shaded.}
\label{fig:ena-sizes}
\end{figure}

% Continues caption on next page. Requires package ccaption.
\begin{figure}[!htb]
  \contcaption{(continued) (G) Values for the Pearson's correlation of the Ena construct fluorescence intensity and its residence time for all movies analyzed. There is no correlation between an Ena construct's intensity and its bound lifetime. \textbf{SEC-MALS performed by Elena Solomaha, University of Chicago Biophysics Core.}}
\end{figure}

Similar to actin elongation rates, average processive run length is also positively correlated with the number of Ena arms (Figure \ref{fig:ena-arms}C-E). Remarkably, although reduced \mytilde10-fold compared to Ena\textsubscript{Tetramer}, Ena\textsubscript{Dimer} does remain processively associated with single filament ($\tau$ = 1.2 s), 2-filament trailing ($\tau$ = 1.5 s), and 3- or more filament trailing ($\tau$ = 2.5 s) barbed ends (Figure \ref{fig:ena-arms}C,E, Table \ref{tab:ena-processive}). Ena\textsubscript{Trimer} has intermediate processivity on single filament ($\tau$ = 5.3 s), 2-filament trailing ($\tau$ = 8.9 s), and 3- or more filament trailing ($\tau$ = 11.2 s) barbed ends (Figure \ref{fig:ena-arms}D,E, Table \ref{tab:ena-processive}). For each construct, the fluorescence intensity is not correlated with run length (Figure \ref{fig:ena-sizes}D-G), indicating that processive activity is not affected by Ena construct multimerization. Ena\textsubscript{Trimer}'s processive run lengths are similar to the residence time of Ena\textsubscript{Tetramer} on single filaments but are not comparably enhanced on trailing barbed ends (Figure \ref{fig:ena-arms}E). Therefore, Ena\textsubscript{Dimer} is sufficient for processive elongation, Ena\textsubscript{Trimer} is necessary for longer processive runs on single filaments, but Ena\textsubscript{Tetramer} is necessary for the longest processive runs on trailing barbed ends of fascin bundles (Figure \ref{fig:ena-arms}E). Interestingly, the avidity effect of multiple filaments in a fascin bundle is apparent even with fewer arms than the wildtype tetramer. The positive correlation between processive elongation and Ena arms is consistent with a recent study on chimeric human VASP with \textit{Dictyostelium} GAB domains on single actin filaments \citep{bruhmann_distinct_2017}. 

\begin{figure}
\centering
\includegraphics[width=5in]{img/ch02/Figure_4_elife.pdf}
\caption[Tetrameric Ena is necessary for proper filopodia density.]{\textbf{Tetrameric Ena is necessary for proper filopodia density.} (A-C) Representative fluorescent micrographs of D16 cells with GFP-actin for (A) control treatment, (B) Ena 3' UTR RNAi, and (C) RNAi with transfection of mCherry-Ena\textsubscript{Tetramer}. White arrows indicate representative filopodia. (D) Boxplot of filopodia density, number of filopodia per micron of cell perimeter, for control cells, Ena 3'UTR RNAi, and RNAi transfected with mCherry-Ena$\Delta$CC-GCN4 (Ena\textsubscript{Dimer}), mCherry-Ena$\Delta$CC-Foldon (Ena\textsubscript{Trimer}), and mCherry-Ena\textsubscript{Tetramer}. n = 3 with at least 10 cells for each experiment. P values (*$<$0.0005). \textbf{D16 cell culture and images captured by Derek Applewhite.}}
\label{fig:ena-cells}
\end{figure}

\subsection{Tetrameric Ena is more efficient at forming filopodia in \textit{Drosophila} culture cells}\label{ena-fly-cells}

Ena\textsubscript{Tetramer} is significantly better at processive actin filament assembly than either Ena\textsubscript{Dimer} or Ena\textsubscript{Trimer}, where Ena\textsubscript{Tetramer} increases the actin elongation rate \mytilde2- to 2.5-fold and remains processively associated with trailing barbed ends of fascin bundles for \mytilde25 sec (Figure \ref{fig:ena-arms}B,E). To determine whether WT Ena\textsubscript{Tetramer} is therefore necessary for proper function in cells, we evaluated the ability of Ena oligomerization constructs to facilitate filopodia in ML-DmD16-c3 \textit{Drosophila} culture cells, derived from third instar larval wing discs (Figure \ref{fig:ena-cells}). We knocked down endogenous Ena with dsRNAi against the 3'UTR and then expressed mCherry-Ena (referred to as mCherry-Ena\textsubscript{Tetramer}), mCherry-Ena$\Delta$CC-GCN4 (referred to as mCherry-Ena\textsubscript{Dimer}) or mCherry-Ena$\Delta$CC-Foldon (referred to as mCherry-Ena\textsubscript{Trimer}) constructs from a constitutive pIZ plasmid (Figure \ref{fig:ena-cells}A-C). The activity of the different Ena constructs was determined by quantifying filopodia density, the number of filopodia per perimeter of the cell (Figure \ref{fig:ena-cells}D). Compared to control cells (0.19 $\pm$ 0.06 filopodia/micron), RNAi treated cells without exogenous Ena have a 2.7-fold decrease in filopodia density (0.07 $\pm$ 0.03 filopodia/micron). Strikingly, mCherry-Ena\textsubscript{Tetramer} forms significantly more filopodia (0.24 $\pm$ 0.05 filopodia/micron) compared to mCherry-Ena\textsubscript{Trimer} (0.15 $\pm$ 0.05 filopodia/micron) and mCherry-Ena\textsubscript{Dimer} (0.15 $\pm$ 0.04 filopodia/micron). There was no correlation between filopodia density and GFP-actin fluorescence or mCherry fluorescence (Figure \ref{fig:ena-expression}). Therefore, Ena tetramers facilitate the production of significantly more filopodia than dimer and trimer constructs following knockdown of endogenous Ena.

\begin{figure}
\centering
\includegraphics[width=6in]{img/ch02/Supp_Figure_3_PNAS.pdf}
\caption[D16 culture cell expression is independent of construct.]{\textbf{D16 culture cell expression is independent of construct.} (A) Dependence of filopodia density on mCherry fluorescence of RNAi treated cells transfected with mCherry-Ena\textsubscript{Dimer}, mCherry-Ena\textsubscript{Trimer}, or mCherry-Ena\textsubscript{Tetramer}. (B) Dependence of filopodia density on the GFP-actin fluorescence intensity for control or RNAi treated control and mCherry-Ena\textsubscript{Dimer}, mCherry-Ena\textsubscript{Trimer}, or mCherry-Ena\textsubscript{Tetramer} transfected cells.}
\label{fig:ena-expression}
\end{figure}

\subsection{Kinetic model of Ena shows a direct correlation between processivity and both bundle size and Ena oligomerization}\label{ena-model}

We observed that Ena's processivity depends on the number of filaments in a fascin bundle (Figure \ref{fig:ena-filaments}D) and number of Ena arms (Figure \ref{fig:ena-arms}E). Therefore, it is likely that the underlying molecular mechanism for Ena's increased processivity on trailing barbed ends depends on Ena's ability to simultaneously bind to an elongating barbed end and sides of filaments via its multiple arms (Figure \ref{fig:ena-bundlers}B). To investigate this avidity effect, we developed a kinetic model of Ena with varying number of arms, $N$, binding bundles composed of varying number of actin filaments, $n$ (Figure \ref{fig:ena-model}A, \ref{fig:ena-si-model}A). 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Figure_5_elife.pdf}
\caption[Kinetic model of Ena/VASP on actin bundles shows processivity positively correlates with both number of Ena arms and bundle size.]{\textbf{Kinetic model of Ena/VASP on actin bundles shows processivity positively correlates with both number of Ena arms and bundle size.} (A) Modeling schematics showing (from left to right). [1] An Ena arm's GAB domain binds the trailing barbed end with binding rate $k^{t}_{on,1}$. [2] Once the GAB domain is bound, the FAB domains from the other arms bind to sides of either the trailing filament ($k^{t}_{on}$) or leading filaments ($k^{l}_{on}$). [3] Arms can be bound to the trailing filament, while others bind leading filaments. (B) Bar graph of the average processive run length as a function of number of Ena arms and bundle size. Error bars, SEM. (C) Heat map showing average Ena run length in the case of 3-filament bundles and four Ena arms, with systematic variations of $k^{l}_{on}$ and $k^{l}_{off}$. Diamond denotes optimized rates for fascin bundles and region within dotted line shows potential rates for $\alpha$-actinin and fimbrin. (D) Average time between binding events ($\tau_{free}^{arm}$) for varying arm number and bundle size. \textbf{Kinetic Model by Harshwardhan Katkar, Tamara Bidone, and Fikret Aydin.}}
\label{fig:ena-model}
\end{figure}

Our model considers binding and unbinding kinetics of all $N$ Ena arms on various binding sites of individual actin filaments in a bundle, which together dictate the kinetics of the Ena “molecule” as a whole (Figure \ref{fig:ena-model}A). An Ena arm initially binds to the trailing barbed end with an on rate of $k_{on,1}^{t}$ and unbinds with an off rate of $k_{off,1}^t$ (Figure \ref{fig:ena-model}A1). The remaining Ena arms are available to bind and unbind to the side of the trailing filament with a rate $k_{on}^{t}$ and $k_{off}^{t}$ or to the side of other filaments in the bundle with a rate $k_{on}^{l}$ and $k_{off}^{l}$ (Figure \ref{fig:ena-model}A2-3). A Monte Carlo algorithm was used to integrate rates of binding and unbinding of Ena arms over time as described in the SI. The model parameter $k_{on,1}^{t}$ was 0.007 s\textsuperscript{-1}, estimated using the TIRFM measured off rate of 0.109 s\textsuperscript{-1} for Ena, and an equilibrium constant of Ena for the barbed end of 0.8 nM \citep{winkelman_ena/vasp_2014}. We therefore considered the local concentration of Ena near the barbed end as 50 pM. The other model parameters were optimized using TIRFM off rates for $N\in(2,3,4)$ and $n\in(1,2,\geq3)$ (Figure \ref{fig:ena-arms}E), as described in the Supplementary Information. 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/Supp_Figure_4_PNAS.pdf}
\caption[Kinetic model can explore different parameter spaces.]{\textbf{Kinetic model can explore different parameter spaces.} (A) Diagram showing the key steps in the algorithm. Nb represents the number of arms bound at time $t$. $\tau_{start}$ and $\tau_{end}$ mark the start and end of Ena "molecule" binding events. The processive run length $\tau$ is estimated as the average of the difference $\tau_{end} - \tau_{start}$ across several events. }
\label{fig:ena-si-model}
\end{figure}

% Continues caption on next page. Requires package ccaption.
\begin{figure}[!htb]
  \contcaption{(continued) (B – F) Heat maps showing average Ena processive run length with systematic variations of $k^{l}_{on}$ and $k^{l}_{off}$. Heat maps of average processive run length for (B) single filaments (C) 2-filament bundles or (D) 4-filament bundles with four Ena arms. These are comparable to Figure \ref{fig:ena-model}C with 3-filament bundles. Heat maps of average processive run length for 3-filament bundles with (E) two or (F) three Ena arms. \textbf{Kinetic Model by Harshwardhan Katkar, Tamara Bidone, and Fikret Aydin.}}
\end{figure}

We used the model to characterize Ena's processive run length at the trailing barbed end. Increasing both the number of filaments in a bundle and the number of Ena arms increases Ena's processive run length, which strongly supports the avidity hypothesis. The modeling results are also in excellent agreement with the trends observed from our TIRFM data (Figure \ref{fig:ena-model}B). Using the model, we tested conditions over a range of both $k_{on}^{l}$ and $k_{off}^{l}$ to mimic $\alpha$-actinin and fimbrin bundles (Figure \ref{fig:ena-bundlers}I), where Ena processivity is not enhanced on trailing barbed ends (Figure \ref{fig:ena-si-model}B-F). The model shows a broad regime that results in the same average processive run length on both leading and trailing barbed ends (Figure \ref{fig:ena-model}C, dashed region). This indicates that differences between bundlers could be due to an effect on Ena's association and dissociation rates caused by differences in how CH domain bundlers and fascin bind F-actin. 

Finally, we used the model to estimate rates of Ena-mediated filament elongation. While at least one Ena arm associates with the barbed end, its other arms undergo binding and dissociation events. When free, an arm can bind G-actin from solution and transfer it to the barbed end. The elongation rate of the Ena bound filament should be proportional to the average time that individual arms are free. From the model, the average time that individual arms remain unbound while the Ena molecule is in the bound state, $\tau_{free}^{arm}$, increases with $N$, and decreases with $n$ (Figure \ref{fig:ena-model}D). This result is consistent with the TIRFM data for the fold increase of actin elongation rate due to Ena on the leading ($n=1$ in the model) and trailing barbed ends ($n>1$ in the model) (Figure \ref{fig:ena-bundlers}K, 3B).

\subsection{Ena binds longer to sides of fascin bundles compared to fimbrin and \texorpdfstring{$\alpha$}{a}-actinin bundles.}

Our kinetic model suggested an area of kinetic rates for Ena binding to the side of a leading filament that could account for the specificity of Ena's enhanced processivity to fascin bundles. To test if there is a difference for Ena binding or dissociating from sides of fascin bundles compared to other bundling proteins we used two approaches, steady-state sedimentation assays and TIRF microscopy.

We first tested if there is a difference in Ena's F-actin binding in the presence and absence of fascin using a bulk sedimentation assay. We used two different truncation constructs to test Ena's F-actin binding affinity, GABFAB and EVH2 (Figure \ref{fig:sedimentation}A). GABFAB contains the GAB and FAB domains and is a monomeric protein while EVH2 contains the GAB, FAB, and CC domains and, therefore, is a tetramer. We saw no significant difference in either the GABFAB (Figure \ref{fig:sedimentation}B-C) or EVH2 F-actin binding (Figure \ref{fig:sedimentation}D-E). These results suggests that there is no difference in FAB binding to the sides of bundled filaments that can be measured in a bulk steady-state assay. 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{img/ch02/sedimentation_thesis.pdf}
\caption[GABFAB and EVH2 bind similarly to single filaments and fascin bundles in bulk sedimentation.]{\textbf{GABFAB and EVH2 bind similarly to single filaments and fascin bundles in bulk sedimentation.} (A) Domain organization for constructs used in sedimentation. G-actin binding domain (G), F-actin binding domain (F), coiled coil region (CC). (B) 15\% SDS-PAGE gel showing supernatant (left) and pellet (right) for the specified combination of 5 $\mu$M GABFAB, 5 $\mu$M actin, and 1 $\mu$M fascin. The reactions were done in duplicate. (C) Percent of GABFAB found in pellet for GABFAB alone, GABFAB with actin, and GABFAB with actin and fascin. Error bars, SEM. n = 8. P values (* $<0.05$). (D) 12.5\% SDS-PAGE gel showing supernatant (left) and pellet (right) for specified combinations of 2.5 $\mu$M EVH2, 5 $\mu$M actin, and 1 $\mu$M fascin. The reactions were done in duplicate. (E) Percent of EVH2 found in pellet for EVH2 alone, EVH2 with actin, and EVH2 with actin and fascin. Error bars, SEM. n = 8. P values (* $<0.05$).}
\label{fig:sedimentation}
\end{figure}

Next, we further analyzed our TIRFM data to measure Ena's affinity for sides of bundles at the resolution that we measure its processivity on barbed ends. We measured the residence time of Ena on sides of single filaments and 2-filament bundles in the presence of fascin, fimbrin, and $\alpha$-actinin (Figure \ref{fig:ena-sidebinding}A-C, Table \ref{tab:ena-p-sidebinding}). Surprisingly, Ena binds longer to the sides of 2-filament bundles only when the bundles are formed by fascin (Figure \ref{fig:ena-sidebinding}D). This suggests that the specificity we see for Ena's enhanced processivity on the barbed end is due to a difference in affinity of Ena's F-actin binding domain for fascin bundles. 

\begin{figure}
\centering
\includegraphics[width=13cm]{img/ch02/side_binding.pdf}
\caption[Ena binds longer on sides of fascin bundles.]{\textbf{Ena binds longer on sides of fascin bundles.} (A-C) Kaplan-Meier curves representing average lifetimes ($\tau$) for 15 pM SNAP(549)-Ena$\Delta$L with (A) 130 nM fascin, (B) 100 nM fimbrin, or (C) 125 nM $\alpha$-actinin on single filaments (red) and 2-filament bundles (blue). Error bars, 95\% CI. $n\geq 58$. (D) Average lifetime for Ena binding to sides of single filaments (red) and 2-filament bundles (blue) shown in A-C with fascin, fimbrin, or $\alpha$-actinin. Error bars, 95\% CI. P values (*$<$0.0001).}
\label{fig:ena-sidebinding}
\end{figure}

\begin{table}[!htb]
\centering
\begin{tabulary}{1.0\textwidth}{C C C C C C}
\toprule 
Bundling protein & Side 1 fil.(s)\textsuperscript{a} & Side 2 fil.(s)\textsuperscript{a} & 1/2 p-value\textsuperscript{b} & Bundling protein\textsuperscript{c} & Comparison p-value \textsuperscript{d} \\
\midrule
Fascin & 0.4\hspace{1.7cm} [0,1.0]\hspace{1.5cm} (125) & 1.9\hspace{1.5cm} [1.4,2.5]\hspace{1.5cm} (161) & ${<0.0001}$ & $\alpha$-actinin & 0.008 \\
Fimbrin & 0.3\hspace{1.7cm} [0,0.6]\hspace{1.5cm} (68) & 0.3\hspace{1.5cm} [0.1,0.5]\hspace{1.5cm} (58) & 0.5 & Fascin & 0.0002 \\
$\alpha$-actinin & 0.3\hspace{1.5cm} [0.2,0.5]\hspace{1.0cm} (129) & 0.7\hspace{1.5cm} [0.4,1.2]\hspace{1.5cm} (174) & 0.2 & Fimbrin & 0.2 \\
\bottomrule
\end{tabulary}
\caption[Side binding residence times and p-values for Ena\textsubscript{Tetramer} with different bundling proteins.]{\textbf{Side binding residence times and p-values for Ena\textsubscript{Tetramer} with different bundling proteins.} \\
\textsuperscript{a} Values of average Ena\textsubscript{Tetramer} lifetime on filament sides (s) [95\%CI] (n) where n is the number of Ena/VASP binding events measured in at least two movies for single filaments or 2-filament bundles.\\
   \textsuperscript{b} Log Rank p-value comparing average lifetime of filament sides for single filaments and 2-filament bundles. \\
   \textsuperscript{c} Bundling protein to compare average lifetime on sides of 2-filament bundles \\
   \textsuperscript{d} Log Rank p-value comparing average lifetime on sides of 2-filament bundles for two specified bundling proteins.}
\label{tab:ena-p-sidebinding}
\end{table}

\section{Discussion}\label{ch02-discussion}

\subsection{Ena's processivity is enhanced specifically on fascin bundles}\label{ena-discuss-specific-fascin}

Ena/VASP proteins are important processive actin elongation factors that are localized to diverse F-actin networks composed of filaments bundled by different crosslinking proteins, including fascin, fimbrin, and $\alpha$-actinin. Previously, we found that Ena takes \mytilde3-fold longer processive runs on trailing barbed ends of fascin-bundled F-actin \citep{winkelman_ena/vasp_2014}. Here we investigated the mechanism and conservation of Ena/VASP's processivity at the barbed end of single filaments and filaments bundled by different crosslinking proteins, as well as the physiological relevance of Ena/VASP tetramerization.

We found that although fly Ena's processivity is enhanced \mytilde3-fold on trailing barbed ends in fascin bundles, there is no processivity enhancement on trailing barbed ends of $\alpha$-actinin or fimbrin bundles (Figure \ref{fig:ena-bundlers}I). Fimbrin and $\alpha$-actinin use two CH domains to bundle F-actin, whereas fascin uses $\beta$-trefoil domains. Though the exact mechanism for Ena's specificity for fascin bundles remains unclear, we suggest several hypotheses. First, fascin could hold the trailing filament in a specific register with respect to the leading filament, allowing for easier Ena/VASP binding. Second, fascin's strong cooperativity \citep{yamakita_phosphorylation_1996,winkelman_fascin-_2016} could promote bundling closer to the growing trailing barbed ends, thereby promoting longer processive runs by keeping trailing barbed ends closer to sides of leading filaments. Third, it is also possible that Ena weakly associates with fascin, although no interaction has yet been detected. If Ena does associate with fascin, it would need to be carefully tuned because a strong interaction could pull Ena from the barbed end. Furthermore, we show that with increasing concentration of fascin that Ena's enhanced processivity is consistent which suggests the absence of a direct interaction (Figure \ref{fig:fascin-range}D).

The fourth hypothesis is based off of a result from our kinetic model, which revealed a broad region of Ena binding kinetics to sides of bundled filaments ($k_{on}^{l}$ and $k_{off}^{l}$) that could explain Ena's lack of enhanced processivity on fimbrin and $\alpha$-actinin bundles (Figure \ref{fig:ena-model}B). We show that in steady-state assays that there is no difference in Ena's affinity for F-actin (Figure \ref{fig:sedimentation}). However, in TIRFM experiments we observe longer residence times on 2-filament fascin bundles compared to fimbrin and $\alpha$-actinin bundles (Figure \ref{fig:ena-sidebinding}D). This difference in dissociation rates could be due to a particular structure of the fascin bundle as suggested previously. Alternatively, it is possible that Ena's on and off rates are affected by competition between Ena and the CH domain bundling proteins for similar binding sites on sides of actin filaments.

Further studies of how fascin forms F-actin networks differently than $\alpha$-actinin and fimbrin will be required to fully elucidate the underlying molecular mechanism. However, this important observation reveals for the first time that bundling proteins and the F-actin networks they form can differentially regulate the activity of processive actin assembly factors, thereby providing a mechanism to allow Ena/VASP proteins to facilitate the assembly of diverse bundled networks with different dynamics in cells. Understanding how different bundling proteins associate with and help form specific F-actin networks in cells will therefore be of critical importance.

\subsection{The mechanism of tetrameric Ena acting on fascin bundles for filopodia formation}\label{filopodia-mechanism}

Given that Ena localizes to filopodia with fascin, lamellipodia with fimbrin, and stress fibers with $\alpha$-actinin, sensitivity to diverse bundles could play an important role in regulating Ena activity in cells. Filopodia are unique amongst these networks with long, straight filaments that emerge from a network capped by capping proteins. Lamellipodia have short, branched filaments and stress fibers are contractile, bipolar networks. Thus, filopodia are the ideal network for enhanced Ena/VASP processivity facilitating elongation of longer filaments that requires stronger competition against capping protein to form a protrusive network. The increased residence time on trailing barbed ends could play a critical role in a feedback mechanism between Ena and fascin in emerging filopodia \citep{winkelman_ena/vasp_2014}. Ena/VASP-associated barbed ends elongate faster, assembling longer actin filaments that contain more fascin binding sites, which subsequently enhance Ena/VASP's processivity. Trailing barbed ends that have longer Ena processive runs can catch up to the leading barbed end, allowing all filaments to reach the same length and resulting in mature filopodia with uniform thickness and aligned barbed ends.
 
\subsection{Avidity promotes enhanced Ena processivity on fascin bundles.}\label{avidity-processivity}

We hypothesize that avidity between multiple actin filaments in a fascin bundle and multiple Ena arms promotes the formation of long filopodia filaments. We investigated the avidity effect by testing how the number of filaments in a fascin bundle and number of Ena arms affects Ena's processive run length. Our results strongly indicate that avidity plays a major role, as there is a \mytilde2-fold increase in Ena's residence time on trailing barbed ends in 2-filament bundles and an additional \mytilde1.5-fold increase on bundles with 3- or more filaments compared to single filament barbed ends (Figure \ref{fig:ena-filaments}B-D). Similarly, the residence time of both VASP and UNC-34 is longer on trailing barbed ends and is correlated with number of actin filaments in a fascin bundle (Figure \ref{fig:ena-filaments}E-G). Furthermore, the residence time of Ena\textsubscript{Trimer} and Ena\textsubscript{Tetramer} is \mytilde4.5- and \mytilde10-fold longer than Ena\textsubscript{Dimer} on fascin bundles with 3- or more filaments (Figure \ref{fig:ena-arms}C-E). A recent study measuring processive elongation using chimeric human VASP with \textit{Dictyostelium} GAB domains on single filaments \citep{bruhmann_distinct_2017} supports our conclusions that enhanced elongation and processive run length are positively correlated with the number of Ena arms. Observing this positive correlation under more physiological conditions, a construct using Ena's unmodified EVH2 domains and on fascin bundles, indicates that these properties are relevant for Ena's activity in cells and specifically for filopodia. 

We further tested the avidity hypothesis by developing a kinetic model that incorporates Ena with differing number of arms binding to single or multiple filaments (Figure \ref{fig:ena-model}). Previous models have focused exclusively on modeling the kinetics of Ena/VASP-mediated barbed end elongation of single actin filaments \citep{hansen_vasp_2010, breitsprecher_molecular_2011, bruhmann_distinct_2017}. As predicted by these models, VASP-mediated single filament elongation rates were shown to increase linearly with the number of VASP arms in solution \citep{breitsprecher_molecular_2011}. However, this model overlooks the binding kinetics of arms that are not associated with the barbed end. Hence, we developed a kinetic model that explicitly incorporates the binding and unbinding rates of each Ena arm on multiple filaments (Figure \ref{fig:ena-model}A). After an Ena arm binds to the barbed end ($k_{on,1}^{t}$), the remaining arm(s) are free to bind to the side of the leading filament(s) ($k_{on}^{l}$) or the trailing filament ($k_{on}^{t}$). We quantified the processive run length for various numbers of bundled filaments and Ena arms. 

The model demonstrates that the avidity effect of Ena emerges from an effective increase in local concentration of F-actin that allows for more FAB binding sites and from multiple Ena arms with available FAB domains. The avidity effect results in longer residence times near the trailing barbed end. Importantly, if an arm dissociates from the trailing barbed end, Ena will continue to processively elongate the barbed end and not diffuse away given that other arms' FAB domains are associated with nearby actin filaments. Furthermore, our model that includes multiple arms binding to multiple actin filaments still has a linear correlation of elongation rates with number of Ena arms on single filaments (Figure \ref{fig:ena-model}D), as predicted by a previous model \citep{breitsprecher_molecular_2011}. The $\tau_{free}^{arm}$ is linear with respect to increasing additional Ena arms on single filaments, but with increasing number of filaments there are diminishing returns by adding more Ena arms. $\tau_{free}^{arm}$ peaks at a tetramer on larger bundles, which gives an additional argument of why a tetramer of Ena/VASP is evolutionarily preferred. We also observe that an Ena tetramer is more efficient at forming filopodia in \textit{Drosophila} culture cells compared to dimer and trimer constructs (Figure \ref{fig:ena-cells}). Since the tetramer has increased residence time on trailing barbed ends and increases actin's elongation rate above the dimer and trimer, this suggests that the tetramer is necessary for proper actin elongation rates and competition with capping protein to allow for the formation of the correct number of filopodia. 

\section{Materials and Methods}\label{ch02-materials-methods}

\subsection{Total internal reflection fluorescence microscopy (TIRFM)}\label{ena-mm-tirf}

TIRFM images were collected at 250 ms-1 s intervals with a cellTIRF 4Line system (Olympus, Center Valley, PA) fitted to an Olympus IX-71 microscope with through-the-objective TIRF illumination and an iXon EMCCD camera (Andor Technology, Belfast, UK). Mg-ATP-actin (15\% Oregon Green or Alexa 488-labeled) was mixed with polymerization TIRF buffer [10 mM imidazole (pH 7.0), 50 mM KCl, 1 mM MgCl\textsubscript{2}, 1 mM EGTA, 50 mM DTT, 0.2 mM ATP, 50 $\mu$M CaCl\textsubscript{2}, 15 mM glucose, 20 $\mu$g/mL catalase, 100 $\mu$g/mL glucose oxidase, and 0.5\% (400 centipoise) methylcellulose] to induce F-actin assembly and any additional actin binding proteins. This mixture was transferred to a flow cell for imaging at room temperature. For two color TIRFM, we cyclically imaged labeled actin (1 frame, 488 nm excitation for 50ms) and SNAP(549)-Ena/VASP (1 frame, 561 nm excitation for 50ms) \citep{winkelman_ena/vasp_2014}.

\subsection{D16 cell culture}\label{ena-mm-cell-culture}

ML-DmD16-c3 (DGRC, Bloomington, IN) cells were cultured in Schneider's Media with 10\% Fetal Bovine Serum (Gibco, Waltham, MA), Anti-Anti (Gibco, Waltham, MA), and 10 $\mu$g/mL recombinant human insulin (Gibco, Waltham, MA), transfected with FugeneHD (Promega, Madison, WI), and imaged on extracellular matrix (ECM) coated glass-bottom dishes after 48–72 hr. ECM was harvested from ML-DmD17-c3 (DGRC, Bloomington, IN) \citep{currie_using_2011}. All imaging was performed on a total internal reflection fluorescence (TIRF) system mounted on an inverted microscope (Ti-E, Nikon, Tokyo, Japan) using a 100X/1.49NA oil immersion TIRF objective driven by Nikon Elements software. Images were captured using an Orca-Flash 4.0 (Hamamatsu, Hamamatsu, Japan) and were processed for brightness and contrast using ImageJ \citep{schneider_nih_2012} analysis. We quantified $>$30 cells using CellGeo \citep{tsygankov_cellgeo:_2014}. Filopodia were quantified with the criteria of $>$0.78 $\mu$m long and $<$0.91 $\mu$m wide. 

\subsection{Plasmid Construction}\label{ena-mm-plasmid}

Enabled (Ena) constructs were prepared by removing the 6x-His tag from the C-terminus of previously described Ena constructs [MBP-SNAP-Ena$\Delta$L or MBP-Ena$\Delta$L] \citep{winkelman_ena/vasp_2014} and insertion into a MBP containing plasmid (pet21A) by standard restriction digest and infusion (Clontech, Mountain View, CA) following PCR amplification (iProof; Bio-Rad, Hercules, California). Ena\textsubscript{Dimer} and Ena\textsubscript{Trimer} constructs were prepared by removing the coiled-coil domain and adding a Foldon domain \citep{guthe_very_2004,papanikolopoulou_formation_2004} [MBP-SNAP-Ena$\Delta$L$\Delta$CC-Foldon] or GCN4 domain \citep{harbury_switch_1993} [MBP-SNAP-Ena$\Delta$L$\Delta$CC-GCN4] from MBP-SNAP-Ena$\Delta$L. UNC-34 was cloned from worm cDNA and inserted into a pet21A vector with MBP-SNAP (New England Biolabs, Ipswich, MA) at XmaI/PacI sites, while also including a flexible linker (GGSGGS) in the forward primer sequence of SNAP constructs. Singed and VASP constructs were cloned from fly and human cDNA libraries, respectively. VASP was inserted into a MBP-SNAP and SNAP containing vector while Singed was inserted into a pGEX KT Ext plasmid containing GST with a Thrombin cleavage site at XbaI/XhoI sites. Plasmids for transfection of mCherry-Ena$\Delta$CC-GCN4 and mCherry-Ena$\Delta$CC-Foldon were cloned into a pIZ-mCherry-Ena \citep{bilancia_enabled_2014} construct using infusion (Clontech, Mountain View, CA). The RNAi was designed using Primer3Plus \citep{untergasser_primer3new_2012} targeting the 3' UTR of \textit{enabled} using forward primer 5' TAATACGACTCACTATAGGGAGACCACGTGATGGCATGTGCATAGGC 3' and reverse primer 5' TAATACGACTCACTATAGGGAGACCACTGCTGAAGACTTGCTGGTTC 3'. The 3'UTR was extracted from w1118 strain fly genome and the DNA region of interest was isolated by PCR amplification and placed in a bluescript SK vector. dsDNA was produced using PCR amplification and dsRNA was produced from the resulting dsDNA using MEGAscript T7 Transcription kit (Invitrogen, Waltham, MA).

\subsection{Protein Expression and Purification}\label{ena-mm-protein-purification}

Recombinant Ena/VASP proteins were purified by expressing in \textit{Escherichia coli} strain BL21-Codon Plus (DE3)-RP (Agilent Technologies, Santa Clara, CA) with 0.25 mM isopropyl $\beta$-D-1-thiogalactopyranoside for 16 h at 16 $^{\circ}$C. Cells were lysed with an Emulsi-Flex-C3 (Avestin, Ottawa, Canada) in extraction buffer [20 mM TRIS-HCl (pH 8.0), 200 mM NaCl, 10\% glycerol, 0.1 mM DTT] with 0.5 $\mu$M PMSF and cOmplete, EDTA-free Protease Inhibitor Cocktail (Roche, Basel, Switzerland) and were clarified. The extract was incubated for 1 h at 4 $^{\circ}$C with amylose resin (New England Biolabs, Ipswich, MA) and the beads were washed with extraction buffer followed by batch elution with elution buffer [20 mM TRIS-HCl (pH 8.0), 200 mM NaCl, 10\% glycerol, 0.1 mM DTT, 40 mM maltose]. Ena/VASP was incubated overnight without and with 1 $\mu$M TEV protease to cleave MBP and filtered on an Superdex 200 10/300 GL or Superose 6 Increase 10/300 GL column (GE Healthcare, Little Chalfont, UK) where they eluted as stable oligomers. Ena/VASP constructs were dialyzed against SNAP buffer [20 mM Hepes (pH 7.4), 200 mM KCl, 0.01\% NaN3, and 10\% Glycerol, and 0.1 mM DTT]. SEC-MALS was performed using DAWN HELEOS II and Optilab T-rEX (Wyatt Technology, Goleta, CA) with a Superdex 200 Increase 10/300 GL column and Akta FPLC (GE Healthcare, Little Chalfont, UK). SEC-MALS data was analyzed using Astra 6.0 (Wyatt Technology, Goleta, CA). SNAP-tagged proteins were labeled with BG-549 (New England Biolabs, Ipswich, MA) following the manufacturers' protocols. Concentrations of SNAP-tagged proteins were determined by densitometry of Coomassie stained bands on SDS/PAGE gels compared with standards or by absorbance at 280 nm. Ena/VASP was flash-frozen in liquid nitrogen and stored at $-80^{\circ}$C. N-terminal SNAP and MBP tags did not affect Ena/VASP's activity. Actin was purified from rabbit skeletal muscle acetone powder (Pel-Freez, Rogers, AR) or self-prepared chicken skeletal muscle acetone powder by a cycle of polymerization and depolymerization and gel filtration \citep{spudich_regulation_1971}. Gel-filtered actin was labeled with Oregon Green \citep{kuhn_real-time_2005} or Alexa 488. Human fascin, human $\alpha$-actinin IV, and \textit{S. pombe} fimbrin were expressed in bacteria and purified as described \citep{vignjevic_formation_2003,skau_fimbrin_2010,li_f-actin_2016}. Singed was purified in the same manner as previously reported for human fascin \citep{vignjevic_formation_2003} except cleavage by thrombin was done off of the column after overnight incubation. 

\subsection{Glass Preparation}\label{ena-mm-glass}

Microscope slides and coverslips (\#1.5; Fisher Scientific, Waltham, MA) were washed for 30 min with acetone and for 10 min with 95\% ethanol, were sonicated for 2 h with Helmanex III detergent (Hellma Analytics, M\"ullheim, Germany), incubated for 2 h with piranha solution (66.6\% H\textsubscript{2}SO\textsubscript{4}, 33.3\% H\textsubscript{2}O\textsubscript{2}), washed with deionized water, and dried. Glass then was incubated for 18 h with 1 mg/mL mPeg-Silane (5,000 MW) in 95\% ethanol, pH 2.0. Parallel strips of double-sided tape were placed on the coverslip to create multiple flow chambers \citep{zimmermann_vitro_2016}. 

\subsection{Calculation of Residence Time and Elongation Rates}\label{ena-mm-resi-elong-measure}

To calculate Ena/VASP's residence time on barbed ends, SNAP(549)-Ena/VASP fluorescent spots associated with the barbed end were manually tracked using MTrackJ \citep{meijering_chapter_2012} in ImageJ. Spots that did not move were not scored, because they were assumed to be adsorbed to the glass. Events that contained joined barbed ends with no clear leading or trailing barbed bend were not included in the average lifetime calculation. Residence times for single SNAP(549)-Ena$\Delta$L tetramers were determined by fitting a Kaplan-Meier \citep{kaplan_nonparametric_1958} survival curve with a single exponential equation, $f(x) = x_{0} * exp^{(-x/\tau)}$ to calculate the average lifetime. Kaplan-Meier survival curves were used to account for processive runs that started before imaging began or ends after imaging terminated. Log rank statistical significance tests were done using Prism 7 (GraphPad Software, San Diego, CA). Barbed end elongation rates were calculated by measuring filament lengths over time with ImageJ software. Multiple filament lengths were plotted over time and the distribution was fit with a linear equation using KaleidaGraph 4.5 (Synergy Software, Reading, PA). To calculate the number of filaments in a bundle the TIRFM movie was used to follow the history of the filaments. This could most accurately differentiate between two-filament bundles and three or more filament bundles. Due to photobleaching of the filaments over time the actin fluorescence was not used to determine the number of filaments within the bundle.

\subsection{Fluorescence Spectroscopy}\label{ena-mm-pyrene}

Bulk actin assembly was measured from the fluorescence of pyrene-actin with a Safire2 or Infinite M200 Pro (Tecan Systems, Inc., M\"annedorf, Switzerland) fluorescent plate reader \citep{neidt_cytokinesis_2008,zimmermann_vitro_2016}. Briefly, unlabeled Mg-ATP-actin was preassembled into seeds for 1 hour by adding 50 mM KCl, 1 mM MgCl\textsubscript{2}, 1 mM EGTA, 10 mM imidazole, pH 7.0. The assay measures the elongation rate of actin by addition of 20\% pyrene-labeled Mg-ATP-actin monomers and actin binding proteins to be assayed. Final protein concentrations are indicated in the figure legends.

\subsection{Sedimentation Assay}\label{ena-mm-sedimentation}
A stock solution of 20 $\mu$M Mg-ATP actin monomers in 10 mM imidazole (pH 7.0), 30 mM KCl, 1 mM MgCl\textsubscript{2}, 1 mM EGTA, 0.5 mM DTT, 0.2 mM ATP, and 90 $\mu$M CaCl\textsubscript{2} were assembled with the addition of any specified bundling proteins and/or Ena construct for 1 h to generate filaments. Any additional specified Ena construct was added and F-actin was then diluted to 5$mu$M final concentration for 20 min at $25^{\circ}$C and spun at 100,000 x g (high-speed) at $24^{\circ}$C. Supernatant and pellets were separated by either 15\% (GABFAB) or 12.5\% (EVH2) SDS–PAGE and stained with Coomassie blue for 30 min, destained for 16 h, and analyzed by densitometry on an Odyssey Infrared Imager (Li-Cor Biosciences, Lincoln, NE).

\section{Acknowledgements}\label{ch02-acknowledgements}

We thank Elena Solomaha of the University of Chicago BioPhysics Core Facility for performing SEC-MALS and Jonathan Winkelman for cloning VASP and UNC-34. We thank Caitlin Anderson, Katie Homa, Cristian Suarez and other members of the D.R.K. laboratory for helpful discussions. We thank the Drosophila Genomics Resource Center, supported by NIH grant 2P40OD010949, for Drosophila cells. This material is based upon work supported by the National Science Foundation Graduate Research Fellowship under Grant No. DGE-1144082 and DGE-1746045 (to A.J.H.), National Institute for Health's Molecular and Cellular Biology Training Grant T32 GM007183 (to A.J.H.), National Institute for Health's Grant RO1 GM079265 (to D.R.K.), Department of Defense Army Research Office's MURI grant W911NF1410403 (to G.A.V. and D.R.K.), and by the University of Chicago Materials Research Science and Engineering Center, funded by National Science Foundation award DMR-1420709 (to G.A.V. and D.R.K.). Acknowledgement is made to the computational resources provided by the Research Computing Center at The University of Chicago.

\section{Supplementary Information}\label{ch02-si}

\subsection{Development of kinetic model}\label{ena-si-develop-model}
In order to test, mechanistically, the hypothesis that avidity of Ena binding multiple actin filaments with multiple arms determines an increase in time spent at the trailing barbed end for fascin-crosslinked bundles, we developed a kinetic model. The model is based on a kinetic Monte Carlo algorithm that at each time step evaluates binding and unbinding probabilities of each Ena arm for each filament and, accordingly, changes the arm "state". The kinetic Monte Carlo scheme is chosen because it can, in principle, give the exact evolution of the system, in terms of bound and unbound states of each Ena arm over time, thus providing a strong approximation of the sequence of events given individual Ena arm's binding and unbinding rates, with respect to individual filaments. The kinetic model used in this work consisted of the following elementary reactions:
\begin{outline}[enumerate]
   \1 Initial binding of an arm of Ena to the barbed end of the trailing filament with a rate of $k_{on,1}^{t}$
   \1 At every subsequent step, binding and unbinding of:
      \2 an arm of Ena to the barbed end of the trailing filament, with rates $k_{on,1}^{t}$ and $k_{off,1}^{t}$
      \2 up to two other arms of Ena to the side of the trailing filament, with rates $k_{on}^{t}$ and $k_{off}^{t}$
      \2 additional arms of Ena beyond three to the side of the trailing filament, with rates $k_{on,4+}^{t}$ and $k_{off,4+}^{t}$
      \2 other arms of Ena to the sides of other filaments in the bundle, with rates $k_{on}^{l}$ and $k_{off}^{l}$
\end{outline}

In summary, once any arm is bound to the barbed end of the trailing filament, the Ena "molecule" is considered to be in the bound state. The Ena molecule unbinds only when none of its arms are bound to any of the filaments in the bundle. Thus, after initiation of the bound state for an Ena molecule, the arm bound to the barbed end can unbind and bind multiple times before the molecule unbinds.
The model was made efficient by only simulating events involving binding and unbinding of the Ena molecule to the barbed end of a trailing filament. Further, we did not intend to calculate the binding rate of the Ena molecule using the model, and instead optimized the model parameters based on TIRFM data (see below) for calculating the unbinding rates of Ena molecules, and predicting the kinetics of individual Ena arms while the molecule was bound. This gave rise to the following possible scenarios while the Ena molecule is in the bound state.
\begin{outline}[enumerate]
   \1 Only one arm is bound to either
      \2 the barbed end
      \2 the side of the trailing filament
      \2 the side of another filament in the bundle
   \1 Two or more arms are bound
      \2 one to the barbed end, others to the side of the same filament
      \2 one to the barbed end, others to the side of another filament
      \2 one to the barbed end, others to the sides of the same and other filament(s)
      \2 some to the side of the same filament and the remaining to the side of another filament
      \2 all to the side of the same filament
      \2 all to the side of another filament
\end{outline}

\subsubsection{Model parameters.}\label{ena-si-parameters}
Since Ena is a homotetramer, all arms in this work are structurally identical to each other. Hence, not all of the eight kinetic rate constants $k_{on,1}^{t}$, $k_{off,1}^{t}$, $k_{on}^{t}$, $k_{off}^{t}$, $k_{on,4+}^{t}$, $k_{off,4+}^{t}$,  $k_{on}^{l}$ and $k_{off}^{l}$ in the model (Figure \ref{fig:ena-model}A) are independent. We set $k_{on,1}^{t}$ = 0.007, estimated using the TIRFM measured off rate of 0.109 s\textsuperscript{-1} for Ena, and an equilibrium constant of Ena for the barbed end of 0.8 nM \citep{winkelman_ena/vasp_2014}. The model assumes that binding rates of the rest of the arms to sides of filaments are identical ($k_{on}^{t} = k_{on,4+}^{t} = k_{on}^{l}$), consistent with the idea that avidity results from binding and unbinding of multiple Ena arms to multiple filaments, rather than from different kinetics of individual arms.  The corresponding unbinding rates were, however, assumed to be different owing to the following reasons. An arm bound to the barbed end interacts with the barbed end of the filament through its GAB domain and potentially its FAB domain, while an arm bound to the side of a filament interacts only through its FAB domain. Thus, $k_{off,1}^{t}$ is considered an independent parameter.  The number of FAB domain binding sites available on the trailing filament can be assumed to be less than those on leading filaments since it is the shortest filament in the bundle. Thus, $k_{off}^{t}$ and $k_{off}^{l}$ are \textit{a priori} considered to be distinct parameters. Our TIRFM data (Figure \ref{fig:ena-arms}B) suggests that the fold increase in processive run length between a trimer and a tetramer binding to a single filament is smaller than the fold increase between a dimer and a trimer. Thus, the fourth arm binding to the same filament is assumed to have different unbinding kinetics represented using the rates $k_{off,4+}^{t}$. This translates to having an upper limit on the number of arms that can simultaneously bind to a given filament.

\subsubsection{Optimization procedure for parameter estimation}\label{ena-si-optimization}

With the above assumptions, the number of undetermined parameters to be estimated reduces to five: $k_{on,1}^{t}$, $k_{on}^{t}$, $k_{off}^{t}$, $k_{off,4+}^{t}$ and $k_{off}^{l}$. These parameters were estimated using all 9 data points for the processive run length data in Figure \ref{fig:ena-arms}B using the Levenberg-Marquardt algorithm implemented in the MATLAB\textsuperscript{\textregistered} function "fsolve". Let $\tau(n,N)$ represent the processive run length of Ena with $N$ arms on the trailing barbed end of a bundle consisting of $n$ actin filaments. The rate ratio vector $y$ is defined as
\begin{equation}\label{eqn:ratio-vector}
\begin{split}
y = [\frac{\tau(1,4)}{\tau(1,2)}, \frac{\tau(1,3)}{\tau(1,2)}, \frac{\tau(2,4)}{\tau(1,4)}, \frac{\tau(2,3)}{\tau(1,3)}, \frac{\tau(2,2)}{\tau(1,2)}, 
 \frac{\tau(4,4)}{\tau(1,4)}, \frac{\tau(4,3)}{\tau(1,3)}, \frac{\tau(4,2)}{\tau(1,2)}] 
\end{split}
\end{equation}
the error was then defined as
\begin{equation}\label{eqn:error}
error=[(y_{model}-y_{TIRFM})/y_{TIRFM}]^{2}
\end{equation}
and minimized iteratively using the five undetermined parameters. For each iteration, the kinetic model was solved for each pair of $N\in(2,3,4)$ and $n\in(1,2,4)$ and the corresponding average processive run length (defined below) was calculated. The TIRFM data for $n\geq 3$ in Figure \ref{fig:ena-arms}E, corresponding to three or more filaments in the bundle, was considered to be equivalent to $n=4$ in the model, consistent with our observation that most bundles in the TIRF data fell between 3 and 5 filaments for an average "large" bundle.

For computational efficiency, we adopted a two-step strategy to obtain the optimum set of parameters. In the first step, we performed error minimization using 50 distinct initial guesses for the parameters and chose six optimized parameter sets with the lowest errors. In the second step, we performed error minimization using 100 sets of initial guesses, each perturbed within $\pm$10\% of the average of these six sets from the first step. The parameter set with the least error was chosen as the final set (Table \ref{tab:model-rates}). A comparison of the rate ratio vectors from the model with corresponding data from TIRFM is shown in Table \ref{tab:model-ratios}. The optimized parameter set was found to predict rate ratios in good agreement with the corresponding ratios from TIRFM data (Figure \ref{fig:ena-arms}E).

\subsubsection{Algorithm}\label{ena-si-algorithm}
Using the values of reaction rates provided in Table \ref{tab:model-rates}, the system evolved using a Monte Carlo algorithm with a constant time-step implemented in MATLAB\textsuperscript{\textregistered}. The states of the arm binding the barbed end and the other arms binding sides of filaments was stored along corresponding columns in a \textit{state} array, with an entry of 0 representing an unbound state and 1 representing a bound state. Each row in the array corresponded to a simulation step. The identity of the filament in the bundle that each Ena arm bound to was stored in a separate \textit{filamentid} array, with filament identities ranging from 1 to $n$. 

The simulation was initialized with all arms of Ena in the unbound state. At each timestep $t + dt$, a reaction move (either binding or unbinding) and the corresponding rate constants were selected depending on the previous state of the system at timestep $t$ (Figure \ref{fig:ena-si-model}A). For example, if the barbed end was bound at timestep t, the unbinding reaction with the rate constant of $k_{off,1}^{t}$ was selected at timestep $t + dt$. $N$ random numbers were generated, one corresponding to each arm, and compared with the rate constant of the selected reaction move. The move was accepted if the random number was less than the corresponding rate constant times $dt$, and the entries \textit{state} and \textit{filamentid} arrays were updated accordingly.

\subsubsection{Model verification and predictions}\label{ena-si-predictions}

The quantities in the model with units of timesteps were converted to real time in seconds by multiplying with a single factor of $5.4374 \times 10^{-3}$ that accounted for the "timescale" and was chosen to exactly match the processive run length for a dimer on a single filament between the model (defined below) and the TIRFM data (leftmost red bar in Figure \ref{fig:ena-arms}E). For computational efficiency, we used $dt = 0.1$ s.
	Assuming that any difference in fascin, $\alpha$-actinin and fimbrin bundles due to spacing between filaments or different interactions should be reflected in the binding and unbinding kinetics, we systematically varied binding/unbinding rates $k_{on}^{l}$ and $k_{off}^{l}$ from 0.002 to 0.026 s\textsuperscript{-1}, keeping other model parameters fixed (Figure \ref{fig:ena-si-model}B-F). For a single filament, the processive run length of an Ena tetramer is independent from $k_{on}^{l}$ and $k_{off}^{l}$ as expected (Figure \ref{fig:ena-si-model}B). With more than one filament, the processive run length increases with $k_{on}^{l}$, for increasing values of $k_{off}^{l}$ below \mytilde0.010 s\textsuperscript{-1} (Figure \ref{fig:ena-si-model}C) or below \mytilde0.026 s\textsuperscript{-1}(Figure \ref{fig:ena-si-model}D). We also systematically changed $k_{on}^{l}$ and $k_{off}^{l}$ using dimers and trimers on 3-filament bundles. Similar to the Ena tetramer, the Ena trimer and dimer showed processive run lengths that increased with $k_{on}^{l}$ for values of $k_{off}^{l}$ below \mytilde0.014 s\textsuperscript{-1} (Figure \ref{fig:ena-si-model}E-F). In the tested range of values for $k_{on}^{l}$ and $k_{off}^{l}$, the maximum run length with dimers is 3 s (Figure \ref{fig:ena-si-model}E) and trimers is 20 s (Figure \ref{fig:ena-si-model}F). Our results show that the processive run length is determined by an interplay between the numbers of arms and filaments, and crosslinker effects on binding rates to sides of leading filaments.
    
\subsubsection{Definitions}\label{ena-si-definitions}

\textbf{Processive run length ($\tau$).} The Ena molecule binding was considered the beginning of a processive run event ($\tau_{start}$, Figure \ref{fig:ena-si-model}A) and unbinding of the Ena molecule ($\tau_{end}$, Figure \ref{fig:ena-si-model}A) denoted the end of a processive run event. The processive run length $\tau$ was calculated by averaging the difference ($\tau_{start}-\tau_{end}$) across all processive run events observed across 56 independent simulation runs, each consisting of a total of $2\times 10^{6}$ timesteps (equivalent to \mytilde10000 seconds).  
For the final data in Figure \ref{fig:ena-model}B, the total number of processive run events used for averaging varied depending on the number of Ena arms and number of filaments in the bundle. Based on the range in our TIRFM data (Figure \ref{fig:ena-arms}E), the number of events were in the range of \mytilde$1.6 \times 10^{5}$ for $(N=4, n=4)$ and $6.8 \times 10^{5}$ for $(N=2, n=1)$. The least number of events used in obtaining data in Figure \ref{fig:ena-model}, \mytilde$4.6 \times 10^{3}$, corresponded to $(N=6, n=6)$.

\textbf{Free arm time ($\tau_{free}^{arm}$).} During each processive run event in the model, individual Ena arms bind to and unbind from filaments independently, but according to their specific rates. The average time between consecutive binding events of an average arm was calculated and denoted as $\tau_{free}^{arm}$. A free Ena arm is available to recruit G-actin from the solution and transfer it to the barbed end with an effective rate that should be independent of the number of filaments in the bundle. Further, since each arm is identical, the effective rate should also be independent of the identity of the arm. It should be noted that in the model Ena arms do not have an identity associated with them and are only used as proxies to obtain statistics related to occupied versus unoccupied states of the barbed end and the sides of filaments. A rapid exchange of an Ena arm bound to the barbed end with an arm bound to the side of a filament is possible but not explicitly accounted for in the model. Thus, though the kinetic model does not explicitly consider filament elongation, $\tau_{free}^{arm}$ is assumed to be approximately proportional to the elongation rate through this implicit effective rate at the resolution of the model.

\clearpage
\subsection{Supplementary Tables}\label{ch02-supplementary-tables}

\begin{table}[!htb]
\centering
\begin{tabular}{ c c c c c c }
\toprule 
$k_{on,1}^{t}$ & $k_{on}^{t}=k_{on,4+}^{t}=k_{on}^{l}$ & $k_{off,1}^{t}$ & $k_{off}^{t}$ & $k_{off,4+}^{t}$ & $k_{off}^{l}$ \\
\midrule
0.007 & 0.0122 & 0.1488 & 0.0049 & 0.0055 & 0.0195 \\
\bottomrule
\end{tabular}
\caption[Final set of rate constants in the kinetic model (s\textsuperscript{-1}).]{\textbf{Final set of rate constants in the kinetic model (s\textsuperscript{-1}).}}
\label{tab:model-rates}
\end{table}

\begin{table}[!htb]
\centering
\begin{tabulary}{1.0\textwidth}{C C C C C C C C C}
\toprule 
Run length ratios & $\tau(1,4)/$ $\tau(1,2)$ & $\tau(1,3)/$ $\tau(1,2)$ & $\tau(2,4)/$ $\tau(1,4)$ & $\tau(2,3)/$ $\tau(1,3)$ & $\tau(2,2)/$ $\tau(1,2)$ & $\tau(4,4)/$ $\tau(1,4)$ & $\tau(4,3)/$ $\tau(1,3)$ & $\tau(4,2)/$ $\tau(1,2)$ \\
\midrule
$y_{model}$ & 8.3240 & 3.3356 & 1.6951 & 1.2305 & 1.0902 & 2.9064 & 1.7222 & 1.2969 \\
$y_{TIRFM}$ & 7.5727 & 4.4074 & 1.8333 & 1.6875 & 1.2489 & 2.8947 & 2.1236 & 2.0825 \\
\bottomrule
\end{tabulary}
\caption[Comparison of processive run length ratios from the model and TIRFM data.]{\textbf{Comparison of processive run length ratios defined in Eqn. \ref{eqn:ratio-vector} from the model and from TIRFM data.}}
\label{tab:model-ratios}
\end{table}